// @gcba/obelisco/dist/organisms/Block/Block.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ButtonBlockProgressBar = exports.ButtonBlockText = exports.ButtonBlockTitle = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Block.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Bloque',
    decorators: [addon_a11y_1.withA11y]
};
var BUTTON_BLOCK = {
    name: 'Iniciar trámite',
    className: 'btn btn-lg btn-primary btn-block',
    url: '#',
    isButton: true
};
var BUTTON_BLOCK_LINK = {
    name: 'Iniciar trámite',
    className: 'btn btn-lg btn-primary btn-block',
    url: '#'
};
var BUTTON_PROGRESS_BAR_BLOCK = {
    name: 'Continuar trámite',
    className: 'btn btn-lg btn-primary btn-block',
    url: '#'
};
var PROGRESS_BAR_BLOCK = {
    type: 'success',
    width: 25
};
exports.ButtonBlockTitle = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-block" },
        react_1.default.createElement(_1.Block, { title: "Inici\u00E1 el tr\u00E1mite y complet\u00E1 todos los pasos", button: BUTTON_BLOCK_LINK }),
        react_1.default.createElement(_1.Block, { title: "Inici\u00E1 el tr\u00E1mite y complet\u00E1 todos los pasos", button: BUTTON_BLOCK_LINK, isLight: true })));
};
exports.ButtonBlockTitle.story = { name: 'Con título' };
exports.ButtonBlockText = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-block" },
        react_1.default.createElement(_1.Block, { title: "Inici\u00E1 el tr\u00E1mite y complet\u00E1 todos los pasos", text: "Si ten\u00E9s un tr\u00E1mite iniciado pod\u00E9s continuarlo desde ac\u00E1.", button: BUTTON_BLOCK }),
        react_1.default.createElement(_1.Block, { title: "Inici\u00E1 el tr\u00E1mite y complet\u00E1 todos los pasos", text: "Si ten\u00E9s un tr\u00E1mite iniciado pod\u00E9s continuarlo desde ac\u00E1.", button: BUTTON_BLOCK, isLight: true })));
};
exports.ButtonBlockText.story = { name: 'Con bajada' };
exports.ButtonBlockProgressBar = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-block" },
        react_1.default.createElement(_1.Block, { title: "Este es tu progreso en el tr\u00E1mite", button: BUTTON_PROGRESS_BAR_BLOCK, blockProgressBar: PROGRESS_BAR_BLOCK }),
        react_1.default.createElement(_1.Block, { title: "Este es tu progreso en el tr\u00E1mite", button: BUTTON_PROGRESS_BAR_BLOCK, blockProgressBar: PROGRESS_BAR_BLOCK, isLight: true })));
};
exports.ButtonBlockProgressBar.story = { name: 'Con barra de progreso' };
