// @gcba/obelisco/dist/components/Button/Button.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.PocoTexto = exports.MuchoTexto = exports.Expandible = exports.Deshabilitados = exports.Tamanos = exports.Activos = exports.ConFoco = exports.Colores = void 0;
// Base
var react_1 = require("react");
require("./Button.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var storybook_addon_xd_designs_1 = require("storybook-addon-xd-designs");
// Components
var utils_1 = require("../utils");
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Botón',
    component: _1.Button,
    decorators: [addon_a11y_1.withA11y, storybook_addon_xd_designs_1.withXD],
    parameters: {
        design: {
            artboardUrl: 'https://xd.adobe.com/view/0623a13b-8fc0-407c-6a4d-c1c24dc2c5fa-7012/screen/d1dd8e3f-6a3b-4a7b-a59e-ba883ca7c684/-TOMOS-BOTONES'
        }
    }
};
exports.Colores = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.buttonTypes.map(function (type) { return (react_1.default.createElement(_1.Button, { key: type, type: type }, type)); })));
};
exports.ConFoco = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.buttonTypes.map(function (type) { return (react_1.default.createElement(_1.Button, { key: type, type: type, className: "focus" }, type)); })));
};
exports.Activos = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.buttonTypes.map(function (type) { return (react_1.default.createElement(_1.Button, { key: type, type: type, className: "active" }, type)); })));
};
exports.Tamanos = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "secondary", size: size }, size)); })));
};
exports.Tamanos.story = { name: 'Tamaños' };
exports.Deshabilitados = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "secondary", disabled: true, size: size }, size)); })));
};
exports.Expandible = function () {
    return (react_1.default.createElement("div", { className: "btn-block-wrapper" },
        react_1.default.createElement(_1.Button, { type: "secondary", block: true }, "Expandible")));
};
exports.MuchoTexto = function () {
    return (react_1.default.createElement("div", { className: "btn btn-success", role: "button", tabIndex: 0 },
        "No est\u00E1 bien que un bot\u00F3n tenga mucho texto, este es un caso extremo que usamos \u00FAnicamente con prop\u00F3sitos de prueba.",
        react_1.default.createElement("br", null),
        "Por favor leer la documentaci\u00F3n correspondiente en los est\u00E1ndares."));
};
exports.PocoTexto = function () {
    return (react_1.default.createElement("div", { className: "btn btn-success", role: "button", tabIndex: 0 }, "Lor"));
};
