// @gcba/obelisco/dist/components/Button/ButtonStates.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ButtonDisabled = exports.ButtonActive = exports.ButtonFocus = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Button.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var utils_1 = require("../utils");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Botones/Estados',
    component: _1.Button,
    decorators: [addon_a11y_1.withA11y]
};
exports.ButtonFocus = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-btn" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, className: "focus" }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-primary focus", href: "#", target: "_blank" }, "Enlace")));
};
exports.ButtonFocus.story = { name: 'Con foco' };
exports.ButtonActive = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-btn" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, className: "active" }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-primary active", href: "#", target: "_blank" }, "Enlace")));
};
exports.ButtonActive.story = { name: 'Activos' };
exports.ButtonDisabled = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-btn" }, utils_1.sizesButton.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.size, type: "secondary", isDisabled: true, size: item.size }, item.name)); })));
};
exports.ButtonDisabled.story = { name: 'Deshabilitados' };
