// @gcba/obelisco/dist/components/Button/Button.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ConIconos = exports.ConBordes = exports.Expandible = exports.Deshabilitados = exports.Tamanos = exports.Activos = exports.ConFoco = exports.Colores = void 0;
// Base
var react_1 = require("react");
require("./Button.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var utils_1 = require("../utils");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Botones',
    component: _1.Button,
    decorators: [addon_a11y_1.withA11y]
};
exports.Colores = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-primary", href: "#", target: "_blank" }, "Enlace")));
};
exports.ConFoco = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, className: "focus" }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-primary focus", href: "#", target: "_blank" }, "Enlace")));
};
exports.ConFoco.story = {
    name: 'Con foco'
};
exports.Activos = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, className: "active" }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-primary active", href: "#", target: "_blank" }, "Enlace")));
};
exports.Tamanos = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        utils_1.sizesButton.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.size, type: "secondary", size: item.size }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-secondary", href: "#", target: "_blank" }, "Enlace")));
};
exports.Tamanos.story = { name: 'Tamaños' };
exports.Deshabilitados = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" }, utils_1.sizesButton.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.size, type: "secondary", disabled: true, size: item.size }, item.name)); })));
};
exports.Expandible = function () {
    return (react_1.default.createElement("div", { className: "btn-block-wrapper" },
        react_1.default.createElement(_1.Button, { type: "secondary", block: true }, "Expandible")));
};
exports.ConBordes = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, outline: true }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-outline-primary", href: "#", target: "_blank" }, "Enlace")));
};
exports.ConBordes.story = {
    name: 'Con bordes'
};
exports.ConIconos = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "btn-wrapper" },
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", size: size, icon: "bxs-car" }, "Bot\u00F3n")); }),
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", outline: true, size: size, icon: "bxs-car" }, "Bot\u00F3n")); }))));
};
exports.ConIconos.story = {
    name: 'Con íconos'
};
