// @gcba/obelisco/dist/components/Button/ContainerButton.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Completo = exports.ConEnlace = exports.ConBoton = void 0;
// Base
var react_1 = require("react");
require("./Button.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Botones/En caja',
    decorators: [addon_a11y_1.withA11y]
};
exports.ConBoton = function () {
    return (react_1.default.createElement("div", { className: "btn-container" },
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-block btn-sm" }, "Bot\u00F3n"))),
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-block" }, "Bot\u00F3n"))),
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-block btn-lg" }, "Bot\u00F3n")))));
};
exports.ConBoton.story = { name: 'Con botón' };
exports.ConEnlace = function () {
    return (react_1.default.createElement("div", { className: "btn-container" },
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("a", { className: "btn btn-primary btn-block btn-sm", href: "#", target: "_blank" }, "Enlace"))),
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("a", { className: "btn btn-primary btn-block", href: "#", target: "_blank" }, "Enlace"))),
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("a", { className: "btn btn-primary btn-block btn-lg", href: "#", target: "_blank" }, "Enlace")))));
};
exports.ConEnlace.story = { name: 'Con enlace' };
exports.Completo = function () {
    return (react_1.default.createElement("div", { className: "btn-container" },
        react_1.default.createElement("div", { className: "card" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("h4", { className: "card-title" }, "Inici\u00E1 el tr\u00E1mite y complet\u00E1 todos los pasos"),
                react_1.default.createElement("p", { className: "card-text" }, "Si ten\u00E9s un tr\u00E1mite iniciado pod\u00E9s continuarlo desde ac\u00E1."),
                react_1.default.createElement("a", { className: "btn btn-primary btn-block", href: "#", target: "_blank" }, "Iniciar tr\u00E1mite")))));
};
exports.Completo.story = {
    name: 'Con título y bajada'
};
