// @gcba/obelisco/dist/components/utils.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.sizeToClass = exports.sizesButton = exports.sizes = exports.alertTypes = exports.buttonTypes = exports.types = void 0;
exports.types = ['primary', 'success', 'danger'];
exports.buttonTypes = [
    { type: 'primary', name: 'Primario' },
    { type: 'secondary', name: 'Secundario' },
    { type: 'success', name: 'Éxito' },
    { type: 'danger', name: 'Error' },
    { type: 'link', name: 'Link' }
];
exports.alertTypes = ['primary', 'success', 'info', 'danger'];
exports.sizes = ['small', 'default', 'large'];
exports.sizesButton = [
    {
        size: 'small',
        name: 'Chico'
    },
    {
        size: 'default',
        name: 'Mediano'
    },
    {
        size: 'large',
        name: 'Grande'
    }
];
exports.sizeToClass = function (size) {
    switch (size) {
        case 'default':
            return '';
        case 'large':
            return 'lg';
        case 'small':
            return 'sm';
    }
};
