// @gcba/obelisco/dist/components/Panel/Panel.constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.LINK_LIST_BUTTONS = exports.LINK_DOWNLOAD_LIST = exports.LINK_LIST = exports.SMALL_LINK_DOWNLOAD = exports.SMALL_PICTURE_XS = exports.SMALL_PICTURE = exports.SMALL_BUTTONS_ARR = exports.VIDEO = exports.PICTURE = exports.LINK_DOWNLOAD_BUTTON_ICON = exports.LINK_DOWNLOAD_BUTTON = exports.LINK_DOWNLOAD = exports.LINK = exports.BUTTONS_ARR = exports.BUTTON_ARR = exports.DESCRIPTION = exports.TITLE = void 0;
/* Large panel */
exports.TITLE = 'Nombre del titular';
exports.DESCRIPTION = 'Descripción de hasta 3 líneas para mantener los estándares de lectura en los componentes de Obelisco.';
exports.BUTTON_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    }
];
exports.BUTTONS_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    },
    {
        name: 'Botón',
        className: 'btn btn-lg btn-secondary card-link'
    }
];
exports.LINK = {
    name: 'Enlace',
    url: '#'
};
exports.LINK_DOWNLOAD = {
    name: 'Descargar',
    className: 'download',
    isDownload: true
};
exports.LINK_DOWNLOAD_BUTTON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg download-link',
    isDownload: true
};
exports.LINK_DOWNLOAD_BUTTON_ICON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg btn-icon card-link',
    icon: 'file_download',
    isDownload: true
};
exports.PICTURE = {
    src: 'panel/hlarreta.jpg',
    alt: 'descripción alternativa'
};
exports.VIDEO = {
    src: 'panel/larreta.mp4',
    alt: 'descripción alternativa'
};
/* Small Panel */
exports.SMALL_BUTTONS_ARR = [
    {
        name: 'Acción 1',
        className: 'btn btn-secondary btn-sm card-link'
    },
    {
        name: 'Acción 2',
        className: 'btn btn-link btn-sm card-link'
    }
];
exports.SMALL_PICTURE = {
    src: 'cards/paseobajo-wide.png',
    alt: 'descripción alternativa',
    className: 'rounded-lg',
    size: 154
};
exports.SMALL_PICTURE_XS = {
    src: 'cards/paseobajo-wide.png',
    alt: 'descripción alternativa',
    className: 'rounded-lg',
    size: 104
};
exports.SMALL_LINK_DOWNLOAD = {
    name: 'Descargar',
    className: 'download',
    url: '#',
    isDownload: true
};
exports.LINK_LIST = {
    listTitle: 'Título de una lista de enlaces',
    links: [
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' }
    ]
};
exports.LINK_DOWNLOAD_LIST = {
    listTitle: 'Título de una lista de enlaces',
    links: [
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true }
    ]
};
exports.LINK_LIST_BUTTONS = [
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    },
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    },
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    }
];
