// @gcba/obelisco/dist/components/Form/Background.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.CasillaDeSeleccion = exports.BloqueDeTexto = exports.ListaDeSeleccion = exports.Texto = void 0;
// Base
var react_1 = require("react");
require("./Form.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
// Config
exports.default = {
    title: 'Componentes|Formulario/Con fondo',
    decorators: [addon_a11y_1.withA11y]
};
exports.Texto = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper bg-light p-4" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "name-input" }, "Nombre"),
            react_1.default.createElement("input", { type: "text", className: "form-control", id: "name-input", name: "name", placeholder: "Ejemplo: Camila" }))));
};
exports.ListaDeSeleccion = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper bg-light p-4" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "lista-de-seleccion" }, "Example select"),
            react_1.default.createElement("select", { className: "custom-select", id: "lista-de-seleccion" },
                react_1.default.createElement("option", null, "1"),
                react_1.default.createElement("option", null, "2")))));
};
exports.ListaDeSeleccion.story = { name: 'Lista de selección' };
exports.BloqueDeTexto = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper bg-light p-4" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "text-area" }, "Bloque de texto"),
            react_1.default.createElement("textarea", { className: "form-control", id: "text-area", rows: 3 }))));
};
exports.CasillaDeSeleccion = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper bg-light p-4" },
        react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "terms-and-conditions", name: "terms-and-conditions", value: "true" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "terms-and-conditions" },
                "Acepto los ",
                react_1.default.createElement("a", { href: "#" }, "t\u00E9rminos y condiciones")))));
};
