// @gcba/obelisco/dist/components/Form/Text.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Deshabilitado = exports.BloqueDeTexto = exports.Fecha = exports.CorreoElectronico = exports.Numero = exports.Texto = void 0;
// Base
var react_1 = require("react");
require("./Form.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
// Config
exports.default = {
    title: 'Componentes|Formulario/Texto',
    decorators: [addon_a11y_1.withA11y]
};
exports.Texto = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "name-input" }, "Nombre"),
            react_1.default.createElement("input", { type: "text", className: "form-control", id: "name-input", name: "name", placeholder: "Ejemplo: Camila" }))));
};
exports.Numero = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "number-input" }, "Edad"),
            react_1.default.createElement("input", { type: "number", className: "form-control", id: "number-input", name: "number", value: "30" }))));
};
exports.Numero.story = { name: 'Número' };
exports.CorreoElectronico = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "email-input" }, "Correo electr\u00F3nico"),
            react_1.default.createElement("input", { type: "email", className: "form-control", id: "email-input", name: "email", "aria-describedby": "email-help", placeholder: "nombre@ejemplo.com" }),
            react_1.default.createElement("small", { id: "email-help", className: "form-text text-muted" }, "Nunca vamos a compartir tu correo electr\u00F3nico con nadie."))));
};
exports.CorreoElectronico.story = { name: 'Correo electrónico' };
exports.Fecha = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "date-input" }, "Fecha"),
            react_1.default.createElement("input", { type: "date", className: "form-control", id: "date-input", name: "date", "aria-describedby": "date-help" }),
            react_1.default.createElement("small", { id: "date-help", className: "form-text text-muted" }, "Ingrese una fecha posterior a hoy."))));
};
exports.BloqueDeTexto = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "text-area" }, "Bloque de texto"),
            react_1.default.createElement("textarea", { className: "form-control", id: "text-area", rows: 3 }))));
};
exports.BloqueDeTexto.story = { name: 'Bloque de texto' };
exports.Deshabilitado = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "name-input" }, "Nombre"),
            react_1.default.createElement("input", { type: "text", className: "form-control", id: "name-input", name: "name", value: "Camila", disabled: true }))));
};
