// @gcba/obelisco/dist/components/Panel/PanelLargeVertical.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.VerticalPanelBackground = exports.VerticalPanelLinkButtonDownload = exports.VerticalPanelLinkDownload = exports.VerticalPanelLink = exports.VerticalPanelButton = exports.VerticalPanelButtons = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var Panel_constants_1 = require("./Panel.constants");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Panel/Grande/Vertical',
    decorators: [addon_a11y_1.withA11y]
};
/* Con Imagen | Video y 2 botones */
exports.VerticalPanelButtons = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, buttons: Panel_constants_1.BUTTONS_ARR })));
};
exports.VerticalPanelButtons.story = { name: 'Con dos botones' };
/* Con Imagen | Video y 1 boton */
exports.VerticalPanelButton = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTON_ARR }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, buttons: Panel_constants_1.BUTTON_ARR })));
};
exports.VerticalPanelButton.story = { name: 'Con un botón' };
/* Con Imagen | Video y enlace */
exports.VerticalPanelLink = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK })));
};
exports.VerticalPanelLink.story = { name: 'Con enlace' };
/* Con Imagen | Video y enlace de descarga */
exports.VerticalPanelLinkDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK_DOWNLOAD }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK_DOWNLOAD })));
};
exports.VerticalPanelLinkDownload.story = { name: 'Con enlace de descarga' };
/* Con Imagen | Video y boton de descarga */
exports.VerticalPanelLinkButtonDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK_DOWNLOAD_BUTTON }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK_DOWNLOAD_BUTTON_ICON })));
};
exports.VerticalPanelLinkButtonDownload.story = { name: 'Con botón de descarga' };
/* Con fondo bris y blanco*/
exports.VerticalPanelBackground = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-panel-vertical" },
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR, bgColor: "light" }),
        react_1.default.createElement(_1.LargePanel, { direction: 'vertical', title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR, bgColor: "white" })));
};
exports.VerticalPanelBackground.story = { name: 'Con fondo' };
