// @gcba/obelisco/dist/organisms/Access/AccessComplete.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Categories = exports.AccessList = exports.Simple = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("../../components/List-group/List-group.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Accesos',
    decorators: [addon_a11y_1.withA11y]
};
var accessList = [
    {
        title: 'Acceso',
        description: 'Descripción (Opcional)'
    },
    {
        title: 'Acceso',
        description: 'Descripción (Opcional)'
    },
    {
        title: 'Acceso',
        description: 'Descripción (Opcional)'
    }
];
var accessCategory = [
    {
        title: 'Categoría 1',
        items: ['Acceso chico', 'Acceso chico']
    },
    {
        title: 'Categoría 2',
        items: ['Acceso chico', 'Acceso chico']
    }
];
exports.Simple = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement(_1.AccessOrganisms, { simpleComponent: true }))); };
exports.AccessList = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement(_1.AccessOrganisms, { accessList: accessList }))); };
exports.AccessList.story = {
    name: 'Con lista de accesos'
};
exports.Categories = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement(_1.AccessOrganisms, { categoryList: accessCategory }))); };
exports.Categories.story = {
    name: 'Con categorías'
};
