// @gcba/obelisco/dist/components/Footer/footer.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SoloRedes = exports.SoloTelefonos = exports.Completo = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración del general del componente
exports.default = {
    title: 'Componentes|Footer',
    decorators: [addon_a11y_1.withA11y]
};
var phones = [
    [102, 'Niñez y Adolescencia'],
    [103, 'Emergencias'],
    [107, 'SAME'],
    [911, 'Policía'],
    [144, 'Violencia de género'],
    [147, 'Atención ciudadana']
];
var social = [
    ['Facebook', 'https://www.facebook.com/GCBA'],
    ['Instagram', 'https://www.instagram.com/buenosaires'],
    ['Twitter', 'https://twitter.com/gcba'],
    ['YouTube', 'https://www.youtube.com/user/GCBA'],
    ['LinkedIn', 'https://ar.linkedin.com/company/gobierno-de-la-ciudad-de-buenos-aires']
];
var PhonesSection = function () { return (react_1.default.createElement("section", null,
    react_1.default.createElement("h4", null, "Tel\u00E9fonos \u00FAtiles"),
    react_1.default.createElement("ul", { className: "list-inline" }, phones.map(function (_a) {
        var number = _a[0], name = _a[1];
        return (react_1.default.createElement("li", { className: "list-inline-item phone-items", key: number },
            react_1.default.createElement("a", { href: "tel:" + number },
                number,
                " - ",
                name)));
    })),
    react_1.default.createElement("a", { href: "https://www.buenosaires.gob.ar/laciudad/telefonosutiles" }, "Ver todos los tel\u00E9fonos"))); };
var SocialSection = function () { return (react_1.default.createElement("section", null,
    react_1.default.createElement("h4", null, "Redes de la ciudad"),
    react_1.default.createElement("ul", { className: "list-inline" }, social.map(function (_a) {
        var network = _a[0], href = _a[1];
        return (react_1.default.createElement("li", { className: "list-inline-item redes-items", key: network },
            react_1.default.createElement("a", { href: href },
                react_1.default.createElement("i", { className: "bx bxl-" + network.toLocaleLowerCase() }),
                network)));
    })))); };
var LegalSection = function () { return (react_1.default.createElement("section", { className: "footer-legal-section" },
    react_1.default.createElement("div", { className: "row align-items-center" },
        react_1.default.createElement("div", { className: "col-12 col-md-5 col-xl-4 mb-4 mb-md-0 footer-content-img" },
            react_1.default.createElement("img", { className: "d-lg-none", src: "footer/ciudad-ba-sm.svg", alt: "Ciudad de Buenos Aires", height: "48" }),
            react_1.default.createElement("img", { className: "d-none d-lg-inline", src: "footer/ciudad-ba.svg", alt: "Ciudad de Buenos Aires", height: "40" }),
            react_1.default.createElement("img", { className: "img-vamos-ba", src: "footer/vamos-ba.svg", alt: "Vamos Buenos Aires" })),
        react_1.default.createElement("div", { className: "col-12 col-md-7 col-xl-8" },
            react_1.default.createElement("ul", { className: "list-inline" },
                react_1.default.createElement("li", { className: "list-inline-item" },
                    react_1.default.createElement("a", { href: "https://boletinoficial.buenosaires.gob.ar" }, "Bolet\u00EDn oficial")),
                react_1.default.createElement("li", { className: "list-inline-item" },
                    react_1.default.createElement("a", { href: "https://www.buenosaires.gob.ar/innovacion/ciudadinteligente/terminos-y-condiciones" }, "T\u00E9rminos y condiciones")),
                react_1.default.createElement("li", { className: "list-inline-item" },
                    react_1.default.createElement("a", { href: "https://www.buenosaires.gob.ar/privacidad" }, "Pol\u00EDtica de privacidad")),
                react_1.default.createElement("li", { className: "list-inline-item" },
                    react_1.default.createElement("a", { href: "https://www.buenosaires.gob.ar/oficiosjudiciales" }, "Oficios judiciales"))))))); };
var LicenseSection = function () { return (react_1.default.createElement("section", null,
    react_1.default.createElement("div", { className: "footer-license-text" },
        "Los contenidos de buenosaires.gob.ar est\u00E1n licenciados bajo ",
        react_1.default.createElement("br", { className: "d-none d-sm-inline" }),
        " Creative Commons Reconocimiento 2.5 Argentina License."))); };
exports.Completo = function () { return (react_1.default.createElement("footer", { className: "main-footer" },
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(PhonesSection, null),
        react_1.default.createElement(SocialSection, null)),
    react_1.default.createElement("hr", { className: "divider" }),
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(LegalSection, null),
        react_1.default.createElement(LicenseSection, null)))); };
exports.SoloTelefonos = function () { return (react_1.default.createElement("footer", { className: "main-footer" },
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(PhonesSection, null)),
    react_1.default.createElement("hr", { className: "divider" }),
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(LegalSection, null),
        react_1.default.createElement(LicenseSection, null)))); };
exports.SoloTelefonos.story = {
    name: 'Solo Teléfonos'
};
exports.SoloRedes = function () { return (react_1.default.createElement("footer", { className: "main-footer" },
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(SocialSection, null)),
    react_1.default.createElement("hr", { className: "divider" }),
    react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement(LegalSection, null),
        react_1.default.createElement(LicenseSection, null)))); };
