// @gcba/obelisco/dist/components/Gallery/GalleryGrid.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnaImagen = exports.DosImagenes = exports.TresImagenes = exports.CuatroImagenes = exports.CincoImagenes = exports.SeisImagenes = exports.SieteImagenes = void 0;
// Base
var react_1 = require("react");
require("./Gallery.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Galería/Grillas',
    decorators: [addon_a11y_1.withA11y]
};
exports.SieteImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-7" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/4.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/5.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/4.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.SieteImagenes.story = {
    name: 'Siete imágenes'
};
exports.SeisImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-6" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/4.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/5.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.SeisImagenes.story = {
    name: 'Seis imágenes'
};
exports.CincoImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-5" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/4.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/5.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.CincoImagenes.story = {
    name: 'Cinco imágenes'
};
exports.CuatroImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-4" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/4.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.CuatroImagenes.story = {
    name: 'Cuatro imágenes'
};
exports.TresImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-3" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/3.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.TresImagenes.story = {
    name: 'Tres imágenes'
};
exports.DosImagenes = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-2" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })),
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/2.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.DosImagenes.story = {
    name: 'Dos imágenes'
};
exports.UnaImagen = function () {
    return (react_1.default.createElement("div", { className: "gallery-grid-container" },
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-1" },
            react_1.default.createElement("div", { className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: "gallery/1.jpg", alt: "Texto alternativo de la imagen" })))));
};
exports.UnaImagen.story = {
    name: 'Una imagen'
};
