// @gcba/obelisco/dist/components/Breadcrumb/breadcrumb.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Mobile = exports.CuatroNiveles = exports.TresNiveles = exports.DosNiveles = exports.UnNivel = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Migas de pan',
    decorators: [addon_a11y_1.withA11y]
};
exports.UnNivel = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", { "aria-label": "breadcrumb" },
        react_1.default.createElement("ol", { className: "breadcrumb" },
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Buenos Aires")))))); };
exports.UnNivel.story = {
    name: 'Un nivel'
};
exports.DosNiveles = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", { "aria-label": "breadcrumb" },
        react_1.default.createElement("ol", { className: "breadcrumb" },
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Buenos Aires")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "P\u00E1gina anterior")))))); };
exports.DosNiveles.story = {
    name: 'Dos niveles'
};
exports.TresNiveles = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", { "aria-label": "breadcrumb" },
        react_1.default.createElement("ol", { className: "breadcrumb" },
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Buenos Aires")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "P\u00E1gina intermedia con mucho mucho texto")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "P\u00E1gina anterior")))))); };
exports.TresNiveles.story = {
    name: 'Tres niveles'
};
exports.CuatroNiveles = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", { "aria-label": "breadcrumb" },
        react_1.default.createElement("ol", { className: "breadcrumb" },
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Buenos Aires")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Intermedia")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Intermedia")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "P\u00E1gina anterior")))))); };
exports.CuatroNiveles.story = {
    name: 'Cuatro niveles'
};
exports.Mobile = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", { "aria-label": "breadcrumb" },
        react_1.default.createElement("ol", { className: "breadcrumb" },
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "Buenos Aires")),
            react_1.default.createElement("li", { className: "breadcrumb-item" },
                react_1.default.createElement("a", { href: "#" }, "P\u00E1gina anterior")))))); };
