// @gcba/obelisco/dist/components/Calendar/CalendarEvents.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.CalendarReferences = exports.CalendarCollapsable = exports.CalendarTwoReferences = exports.CalendarLink = exports.CalendarPredeterminate = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
//component y constants
var index_1 = require("./index");
var Calendar_constants_1 = require("./Calendar.constants");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Calendario / Con eventos',
    decorators: [addon_a11y_1.withA11y]
};
exports.CalendarPredeterminate = function () {
    return react_1.default.createElement(index_1.CalendarMonth, { monthNumber: 1, start: 1, activeDays: Calendar_constants_1.ACTIVE_DAYS_DEFAULT });
};
exports.CalendarPredeterminate.story = { name: 'Predeterminado' };
exports.CalendarLink = function () {
    return react_1.default.createElement(index_1.CalendarMonth, { monthNumber: 1, start: 1, activeDays: Calendar_constants_1.ACTIVE_DAYS_DEFAULT, isLinked: true });
};
exports.CalendarLink.story = { name: 'Con enlace' };
exports.CalendarTwoReferences = function () {
    return react_1.default.createElement(index_1.CalendarMonth, { monthNumber: 1, start: 1, activeDays: Calendar_constants_1.ACTIVE_DAYS_JANUARY_TWO_REFERENCES });
};
exports.CalendarTwoReferences.story = { name: 'Con dos referencias' };
exports.CalendarCollapsable = function () {
    return (react_1.default.createElement(index_1.CalendarMonth, { monthNumber: 1, start: 1, activeDays: Calendar_constants_1.ACTIVE_DAYS_JANUARY_COLLAPSED, isCollapsed: true }));
};
exports.CalendarCollapsable.story = { name: 'Con colapsable' };
exports.CalendarReferences = function () {
    return (react_1.default.createElement(index_1.CalendarMonth, { monthNumber: 1, start: 1, activeDays: Calendar_constants_1.ACTIVE_DAYS_JANUARY_REFERENCES, isCollapsed: true }));
};
exports.CalendarReferences.story = { name: 'Con referencias cromáticas' };
