// @gcba/obelisco/dist/components/Alert/Alert.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.List = exports.ListLinksExample = exports.SimpleHiperlink = exports.SimpleHighlight = exports.SimpleClose = exports.Simple = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Alert.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var index_1 = require("./index");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Alerta',
    decorators: [addon_a11y_1.withA11y]
};
exports.Simple = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.Alert, { arrayAlerts: index_1.alertTypes })));
};
exports.SimpleClose = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.Alert, { arrayAlerts: index_1.alertTypes, isDismissible: true })));
};
exports.SimpleClose.story = {
    name: 'Con cierre'
};
exports.SimpleHighlight = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.Alert, { arrayAlerts: index_1.alertHighlight })));
};
exports.SimpleHighlight.story = {
    name: 'Con destacado'
};
exports.SimpleHiperlink = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.Alert, { arrayAlerts: index_1.alertLink })));
};
exports.SimpleHiperlink.story = {
    name: 'Con hipervínculo'
};
exports.ListLinksExample = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.ListLinks, { type: "danger", text: "<strong>Este es un destacado de una alerta de error.</strong> Esta es la descripci\u00F3n de una alerta de error que continua al texto destacado.", textLink: "Ancla al error", isListLinks: true })));
};
exports.ListLinksExample.story = { name: 'Con lista de enlaces' };
exports.List = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-alert" },
        react_1.default.createElement(_1.ListLinks, { type: "info", text: "<strong>Este es un destacado de una alerta de informaci\u00F3n.</strong> Esta es la descripci\u00F3n de una alerta de informaci\u00F3n que continua al texto destacado.", textLink: "Texto descriptivo", isListLinks: false })));
};
exports.List.story = { name: 'Con lista descriptiva' };
