// @gcba/obelisco/dist/components/ListSteps/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListSteps = exports.LISTSTEPS_CONTENT_LIGHT_ITEMS = exports.LISTSTEPS_CONTENT_LARGE = exports.LISTSTEPS_CONTENT = void 0;
var react_1 = __importDefault(require("react"));
// Components
var utils_1 = require("../utils");
var classnames_1 = __importDefault(require("classnames"));
function createList(size, isTextLarge, startLightItem) {
    var list = [];
    for (var i = 1; i <= size; i++) {
        var step = {
            title: "Instrucci\u00F3n " + i,
            description: isTextLarge
                ? "En esta descripci\u00F3n de la instrucci\u00F3n " + i + " se muestran los detalles principales de las acciones que deber\u00E1 tomar el usuario luego. En esta descripci\u00F3n de la instrucci\u00F3n " + i + " se muestran los detalles principales de las acciones que deber\u00E1 tomar el usuario luego."
                : "En esta descripci\u00F3n de la instrucci\u00F3n " + i + " se muestran los detalles principales de las acciones que deber\u00E1 tomar el usuario luego."
        };
        startLightItem && i >= startLightItem && (step.isLightItem = true);
        list.push(step);
    }
    return list;
}
exports.LISTSTEPS_CONTENT = createList(3);
exports.LISTSTEPS_CONTENT_LARGE = createList(3, true);
exports.LISTSTEPS_CONTENT_LIGHT_ITEMS = createList(5, false, 4);
exports.ListSteps = function (props) {
    var content = props.content, size = props.size, isLight = props.isLight, isNumbered = props.isNumbered;
    var ListStepsClasses = classnames_1.default('list-steps', size && utils_1.sizeToClass(size) && "list-steps-" + utils_1.sizeToClass(size), { 'list-steps-numbers': isNumbered }, { 'list-steps-light': isLight });
    return (react_1.default.createElement("ul", { className: ListStepsClasses.trim() }, content.map(function (_a, index) {
        var title = _a.title, description = _a.description, isLightItem = _a.isLightItem;
        return (react_1.default.createElement("li", { className: "list-steps-item " + (isLightItem ? "list-steps-item-light" : ''.trim()), key: index },
            react_1.default.createElement("h4", null, title),
            description && react_1.default.createElement("p", null, description)));
    })));
};
