// @gcba/obelisco/dist/components/ListSteps/ListSteps.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListStepsNumbers = exports.ListStepsBorder = exports.ListStepsSizes = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./ListSteps.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Listas/Lista de pasos',
    decorators: [addon_a11y_1.withA11y]
};
exports.ListStepsSizes = function () { return (react_1.default.createElement("div", { className: "storybook__wrapper-list-steps" },
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT_LARGE, size: "large" }),
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT }),
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, size: "small" }))); };
exports.ListStepsSizes.story = { name: 'Tamaños' };
exports.ListStepsBorder = function () { return (react_1.default.createElement("div", { className: "storybook__wrapper-list-steps" },
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, size: "large", isLight: true }),
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, isLight: true }),
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, size: "small", isLight: true }))); };
exports.ListStepsBorder.story = { name: 'Con borde' };
exports.ListStepsNumbers = function () { return (react_1.default.createElement("div", { className: "storybook__wrapper-list-steps" },
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, isNumbered: true }),
    react_1.default.createElement(_1.ListSteps, { content: _1.LISTSTEPS_CONTENT, isNumbered: true, isLight: true }))); };
exports.ListStepsNumbers.story = { name: 'Con número' };
//A futuro, tal vez se incluya la version de Pasos completados
/* export const ListStepsComplete = (): JSX.Element => (
  <div className="storybook__wrapper-list-steps">
    <ListSteps content={LISTSTEPS_CONTENT_LIGHT_ITEMS} isNumbered={true} />
  </div>
);
ListStepsComplete.story = { name: 'Con pasos completados' }; */
