// @gcba/obelisco/dist/components/Table/Table.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Seleccionable = exports.TodoJunto = exports.Resaltable = exports.Intercalada = exports.AlineaciónVertical = exports.AlineaciónHorizontal = exports.ConBordes = exports.Normal = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Tablas',
    component: _1.Table,
    decorators: [addon_a11y_1.withA11y]
};
exports.Normal = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ] }));
};
exports.ConBordes = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], bordered: true }));
};
exports.AlineaciónHorizontal = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', rightAligned('Cantidad')], body: [
            ['1', 'Juan', rightAligned('1.234.567')],
            ['2', 'María', rightAligned('34.567')],
            ['3', 'Esther', rightAligned('82.428,53')],
            ['4', 'José', rightAligned('99.131.413')]
        ], bordered: true }));
};
exports.AlineaciónVertical = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', rightAligned('Cantidad')], body: [
            [SmallContent, 'Juan', rightAligned('1.234.567')],
            [SmallContent, 'María', rightAligned('34.567')]
        ], className: "table-align-bottom", bordered: true }));
};
exports.Intercalada = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], striped: true }));
};
exports.Resaltable = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], hoverable: true }));
};
exports.TodoJunto = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], bordered: true, striped: true, hoverable: true }));
};
exports.Seleccionable = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], selectable: true }));
};
var SmallContent = (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("h5", null, "T\u00EDtulo"),
    react_1.default.createElement("p", { className: "mb-0" },
        "Lorem ipsum dolor sit amet,",
        react_1.default.createElement("br", null),
        "repellendus ad est, laborum eum reiciendis quam.")));
var rightAligned = function (content) {
    return {
        content: content,
        props: { className: 'text-right' }
    };
};
