// @gcba/obelisco/dist/components/Badge/Badge.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Link = exports.Colores = exports.Multiple = exports.Simple = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Etiqueta',
    component: _1.Badge,
    decorators: [addon_a11y_1.withA11y]
};
exports.Simple = function () { return react_1.default.createElement(_1.Badge, { text: "Etiqueta 1", className: "badge-secondary" }); };
exports.Multiple = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 1", className: "badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 2", className: "badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 3", className: "badge-secondary" }))); };
exports.Multiple.story = {
    name: 'Múltiple'
};
exports.Colores = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 1", className: "badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 2", className: "badge-danger" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 3", className: "badge-success" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 4", className: "badge-info" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 5", className: "badge-primary" }))); };
exports.Link = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 1", href: "#", className: "badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 2", href: "#", className: "hover badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 3", href: "#", className: "focus hover badge-secondary" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 4", href: "#", className: "focus badge-secondary" }))); };
