// @gcba/obelisco/dist/components/Badge/Badge.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Link = exports.Multiple = exports.Simple = void 0;
// Base
var react_1 = require("react");
// Component
var _1 = require(".");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración del general del componente
exports.default = {
    title: 'Componentes|Etiqueta',
    decorators: [addon_a11y_1.withA11y],
    component: _1.Badge
};
exports.Simple = function () { return react_1.default.createElement(_1.Badge, { text: "Etiqueta 1" }); };
exports.Multiple = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 1" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 2" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 3" }))); };
exports.Link = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 1", href: "#" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 2", href: "#", className: "active" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 3", href: "#", className: "focus active" }),
    react_1.default.createElement(_1.Badge, { text: "Etiqueta 4", href: "#", className: "focus" }))); };
