// @gcba/obelisco/dist/organisms/Header/Header.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SearchOneLine = exports.SearchLoginTwoLines = exports.SearchLoginOneLine = exports.SearchNoLoginTwoLines = exports.SearchNoLoginOneLine = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Header.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Component and Constants
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Header (encabezado)/Con buscador',
    decorators: [addon_a11y_1.withA11y]
};
exports.SearchNoLoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasLogin: true, hasSearch: true, sections: _1.SECTIONS_FIRST_LINE })));
};
exports.SearchNoLoginOneLine.story = {
    name: 'Deslogueado en 1 línea'
};
exports.SearchNoLoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasLogin: true, hasSearch: true, sections: _1.SECTIONS_SECOND_LINE, isSectionExtended: true })));
};
exports.SearchNoLoginTwoLines.story = {
    name: 'Deslogueado en 2 líneas'
};
exports.SearchLoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasProfile: true, hasSearch: true, sections: _1.SECTIONS_FIRST_LINE })));
};
exports.SearchLoginOneLine.story = {
    name: 'Logueado en 1 línea'
};
exports.SearchLoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasProfile: true, hasSearch: true, sections: _1.SECTIONS_SECOND_LINE, isSectionExtended: true })));
};
exports.SearchLoginTwoLines.story = {
    name: 'Logueado en 2 líneas'
};
exports.SearchOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasSearch: true, sections: _1.SECTIONS_SEARCH })));
};
exports.SearchOneLine.story = {
    name: 'Sin logueo'
};
