// @gcba/obelisco/dist/organisms/Header/Header.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SearchOneLine = exports.SearchLoginTwoLines = exports.SearchLoginOneLine = exports.SearchNoLoginTwoLines = exports.SearchNoLoginOneLine = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Header.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Constants
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Header (encabezado)/Con buscador',
    decorators: [addon_a11y_1.withA11y]
};
exports.SearchNoLoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement("header", { className: "navbar navbar-light navbar-expand-lg" },
            react_1.default.createElement("div", { className: "container header-container" },
                _1.LOGO,
                react_1.default.createElement(_1.Login, { isMobile: true }),
                _1.BUTTON_TOGGLER,
                react_1.default.createElement("div", { className: "collapse navbar-collapse", id: "navbarContent" },
                    react_1.default.createElement("div", { className: "navbar-content" },
                        react_1.default.createElement(_1.NavSection, null,
                            _1.DROPDOWN_ITEM,
                            _1.NAV_ITEM,
                            _1.DROPDOWN_ITEM),
                        _1.SEARCH,
                        react_1.default.createElement(_1.Login, null)))),
            react_1.default.createElement("div", { className: "header-backdrop" }))));
};
exports.SearchNoLoginOneLine.story = {
    name: 'Deslogueado en 1 línea'
};
exports.SearchNoLoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement("header", { className: "navbar navbar-light navbar-expand-lg" },
            react_1.default.createElement("div", { className: "container header-container" },
                _1.LOGO,
                react_1.default.createElement(_1.Login, { isMobile: true }),
                _1.BUTTON_TOGGLER,
                react_1.default.createElement("div", { className: "collapse navbar-collapse", id: "navbarContent" },
                    react_1.default.createElement("div", { className: "navbar-content" },
                        _1.SEARCH,
                        react_1.default.createElement(_1.Login, null)),
                    react_1.default.createElement(_1.NavSection, { isExtended: true },
                        _1.DROPDOWN_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM,
                        _1.DROPDOWN_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM))),
            react_1.default.createElement("div", { className: "header-backdrop" }))));
};
exports.SearchNoLoginTwoLines.story = {
    name: 'Deslogueado en 2 líneas'
};
exports.SearchLoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement("header", { className: "navbar navbar-light navbar-expand-lg" },
            react_1.default.createElement("div", { className: "container header-container" },
                _1.LOGO,
                _1.BUTTON_TOGGLER,
                react_1.default.createElement("div", { className: "collapse navbar-collapse", id: "navbarContent" },
                    react_1.default.createElement("div", { className: "navbar-content" },
                        react_1.default.createElement(_1.NavSection, null,
                            _1.DROPDOWN_ITEM,
                            _1.NAV_ITEM,
                            _1.DROPDOWN_ITEM),
                        _1.SEARCH,
                        _1.USER),
                    _1.USER_MOBILE)),
            react_1.default.createElement("div", { className: "header-backdrop" }))));
};
exports.SearchLoginOneLine.story = {
    name: 'Logueado en 1 línea'
};
exports.SearchLoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement("header", { className: "navbar navbar-light navbar-expand-lg" },
            react_1.default.createElement("div", { className: "container header-container" },
                _1.LOGO,
                _1.BUTTON_TOGGLER,
                react_1.default.createElement("div", { className: "collapse navbar-collapse", id: "navbarContent" },
                    react_1.default.createElement("div", { className: "navbar-content" },
                        _1.SEARCH,
                        _1.USER),
                    react_1.default.createElement(_1.NavSection, { isExtended: true },
                        _1.DROPDOWN_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM,
                        _1.DROPDOWN_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM,
                        _1.NAV_ITEM),
                    _1.USER_MOBILE)),
            react_1.default.createElement("div", { className: "header-backdrop" }))));
};
exports.SearchLoginTwoLines.story = {
    name: 'Logueado en 2 líneas'
};
exports.SearchOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement("header", { className: "navbar navbar-light navbar-expand-lg" },
            react_1.default.createElement("div", { className: "container header-container" },
                _1.LOGO,
                _1.BUTTON_TOGGLER,
                react_1.default.createElement("div", { className: "collapse navbar-collapse", id: "navbarContent" },
                    react_1.default.createElement("div", { className: "navbar-content" },
                        react_1.default.createElement(_1.NavSection, null,
                            _1.DROPDOWN_ITEM,
                            _1.NAV_ITEM,
                            _1.DROPDOWN_ITEM,
                            _1.NAV_ITEM),
                        _1.SEARCH))),
            react_1.default.createElement("div", { className: "header-backdrop" }))));
};
exports.SearchOneLine.story = {
    name: 'Sin logueo'
};
