// @gcba/obelisco/dist/organisms/Header/HeaderWithoutSearch.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.OneLine = exports.LoginTwoLines = exports.LoginOneLine = exports.NoLoginTwoLines = exports.NoLoginOneLine = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Header.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Component and Constants
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Header (encabezado)/Sin buscador',
    decorators: [addon_a11y_1.withA11y]
};
exports.NoLoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasLogin: true, sections: _1.SECTIONS_FIRST_LINE })));
};
exports.NoLoginOneLine.story = {
    name: 'Deslogueado en 1 línea'
};
exports.NoLoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasLogin: true, sections: _1.SECTIONS_SECOND_LINE, isSectionExtended: true })));
};
exports.NoLoginTwoLines.story = {
    name: 'Deslogueado en 2 líneas'
};
exports.LoginOneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasProfile: true, sections: _1.SECTIONS_FIRST_LINE })));
};
exports.LoginOneLine.story = {
    name: 'Logueado en 1 línea'
};
exports.LoginTwoLines = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { hasProfile: true, sections: _1.SECTIONS_SECOND_LINE, isSectionExtended: true })));
};
exports.LoginTwoLines.story = {
    name: 'Logueado en 2 líneas'
};
exports.OneLine = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-header" },
        react_1.default.createElement(_1.Header, { sections: _1.SECTIONS_NO_SEARCH })));
};
exports.OneLine.story = {
    name: 'Sin logueo'
};
