// @gcba/obelisco/dist/styles/Typography/Typography.stories.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
    Object.defineProperty(o, "default", { enumerable: true, value: v });
}) : function(o, v) {
    o["default"] = v;
});
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (k !== "default" && Object.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
    __setModuleDefault(result, mod);
    return result;
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.OrderList = exports.List = exports.Quote = exports.Transformations = exports.Epigraph = exports.Text = exports.Subtitle = exports.Headings = void 0;
// Base
var react_1 = __importStar(require("react"));
require("./Typography.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Estilos|Tipografía',
    decorators: [addon_a11y_1.withA11y]
};
exports.Headings = function () {
    var _a = react_1.useState(false), isMobile = _a[0], setIsMobile = _a[1];
    react_1.useEffect(function () {
        function handleResize() {
            setIsMobile(window.innerWidth < 592);
        }
        window.addEventListener('resize', handleResize);
        handleResize();
        return function () { return window.removeEventListener('resize', handleResize); };
    }, []);
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("h1", null, "H1 - Nunito Bold 37px"),
        react_1.default.createElement("h2", null, "H2 - Nunito Semibold 28px"),
        react_1.default.createElement("h3", null, "H3 - Nunito Semibold 21px"),
        react_1.default.createElement("h4", null,
            "H4 - Nunito ",
            isMobile ? 'Semibold' : 'Bold',
            " 19px"),
        react_1.default.createElement("h5", null, "H5 - Nunito Semibold 17px"),
        react_1.default.createElement("h6", null, "H6 - Open Sans Bold 16px")));
};
exports.Headings.story = { name: 'Títulos' };
exports.Subtitle = function () {
    return react_1.default.createElement("p", { className: "lead" }, "Open Sans Regular 19px");
};
exports.Subtitle.story = { name: 'Subtítulo' };
exports.Text = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null, "Open Sans Regular 16px"),
        react_1.default.createElement("small", null, "Open Sans Regular 13px")));
};
exports.Text.story = { name: 'Párrafo' };
exports.Epigraph = function () {
    return react_1.default.createElement("p", { className: "text-xs" }, "Open Sans Regular 11.7px");
};
exports.Epigraph.story = {
    name: 'Epígrafe'
};
exports.Transformations = function () {
    return (react_1.default.createElement("p", null,
        "Nuestro sistema de ",
        react_1.default.createElement("strong", null, "est\u00E1ndares"),
        " brinda un ",
        react_1.default.createElement("em", null, "conjunto de herramientas"),
        " y componentes colaborativos",
        react_1.default.createElement("br", null),
        "y ",
        react_1.default.createElement("small", null, "accesibles"),
        " que define procesos para que los equipos internos y externos puedan trabajar",
        react_1.default.createElement("br", null),
        "alineadamente. De esta forma, ayudamos a crear ",
        react_1.default.createElement("mark", null, "servicios"),
        " consistent",
        react_1.default.createElement("sup", null, "es"),
        " para mejorar",
        react_1.default.createElement("br", null),
        " la experiencia digit",
        react_1.default.createElement("sub", null, "al"),
        " de vecinos y vecina",
        react_1.default.createElement("kbd", null, "s"),
        "."));
};
exports.Transformations.story = {
    name: 'Transformaciones'
};
exports.Quote = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-bloquote" },
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Este es un apartado del texto que se quiere destacar por su importancia dentro del contenido de la p\u00E1gina.")),
        react_1.default.createElement("br", null),
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Este es un apartado del texto que se quiere destacar por su importancia dentro del contenido de la p\u00E1gina."),
            react_1.default.createElement("small", null, "Nombre del autor de la cita"))));
};
exports.Quote.story = {
    name: 'Citas'
};
exports.List = function () {
    return (react_1.default.createElement("ul", null,
        react_1.default.createElement("li", null, "Esto es el texto de una lista"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista")));
};
exports.List.story = {
    name: 'Listas'
};
exports.OrderList = function () {
    return (react_1.default.createElement("ol", null,
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada")));
};
exports.OrderList.story = {
    name: 'Lista ordenada'
};
