// @gcba/obelisco/dist/organisms/Table/Table.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Responsive = exports.Seleccionable = exports.TodoJunto = exports.Resaltable = exports.Intercalada = exports.AlineaciónVertical = exports.AlineaciónHorizontal = exports.ConBordes = exports.Normal = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Table.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Tablas',
    component: _1.Table,
    decorators: [addon_a11y_1.withA11y]
};
exports.Normal = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ] }));
};
exports.ConBordes = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], bordered: true }));
};
exports.ConBordes.story = {
    name: 'Con bordes'
};
exports.AlineaciónHorizontal = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', rightAligned('Cantidad')], body: [
            ['1', 'Juan', rightAligned('1.234.567')],
            ['2', 'María', rightAligned('34.567')],
            ['3', 'Esther', rightAligned('82.428,53')],
            ['4', 'José', rightAligned('99.131.413')]
        ], bordered: true }));
};
exports.AlineaciónHorizontal.story = {
    name: 'Alineación horizontal'
};
exports.AlineaciónVertical = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', rightAligned('Cantidad')], body: [
            [SmallContent, 'Juan', rightAligned('1.234.567')],
            [SmallContent, 'María', rightAligned('34.567')]
        ], className: "table-align-bottom", bordered: true }));
};
exports.AlineaciónVertical.story = {
    name: 'Alineación vertical'
};
exports.Intercalada = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], striped: true }));
};
exports.Resaltable = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], hoverable: true }));
};
exports.TodoJunto = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], bordered: true, striped: true, hoverable: true }));
};
exports.TodoJunto.story = {
    name: 'Todo junto'
};
exports.Seleccionable = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Encabezado', 'Otro más'], body: [
            ['1', 'Juan', 'Alberto'],
            ['2', 'María', 'Julia'],
            ['3', 'Esther', 'Fermin'],
            ['4', 'José', 'Juana']
        ], selectable: true }));
};
var SmallContent = (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("h5", null, "T\u00EDtulo"),
    react_1.default.createElement("p", { className: "mb-0" },
        "Esto es un p\u00E1rrafo,",
        react_1.default.createElement("br", null),
        "esto es otro p\u00E1rrafo.")));
var rightAligned = function (content) {
    return {
        content: content,
        props: { className: 'text-right' }
    };
};
exports.Responsive = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-table-responsive" },
        react_1.default.createElement("div", { className: "responsive-scroll", tabIndex: 0 },
            react_1.default.createElement(_1.Table, { head: ['#', 'Nombre', 'Apellido', 'Dirección', 'Ciudad', 'Comuna', 'Provincia'], body: [
                    ['1', 'Juan', 'Alberto', 'Av. Santa Fe 1234', 'CABA', '1', 'Buenos Aires'],
                    ['2', 'María', 'Julia', 'Av. Santa Fe 1234', 'CABA', '1', 'Buenos Aires'],
                    ['3', 'Esther', 'Fermin', 'Av. Santa Fe 1234', 'CABA', '1', 'Buenos Aires']
                ], bordered: true }))));
};
