// @gcba/obelisco/dist/organisms/Table/Table.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.CheckboxAndText = exports.StripedTable = exports.NormalTable = exports.NoBorder = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Table.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Tablas',
    component: _1.Table,
    decorators: [addon_a11y_1.withA11y]
};
exports.NoBorder = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-table-responsive" },
        react_1.default.createElement("div", { className: "responsive-scroll", tabIndex: 0 },
            react_1.default.createElement("table", { className: "table table-borderless" },
                react_1.default.createElement("thead", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("th", { scope: "col", className: "tb-number" }, "N\u00FAmero"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-link" }, "Enlace"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-button" }, "Bot\u00F3n"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-check" }, "Checkbox"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-tag" }, "Etiqueta"))),
                react_1.default.createElement("tbody", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "1"),
                        react_1.default.createElement("td", null, "Juan"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" }, "Redacci\u00F3n"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "2"),
                        react_1.default.createElement("td", null, "Mar\u00EDa"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-design-input", name: "skills", value: "design" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-design-input" }, "Dise\u00F1o"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "3"),
                        react_1.default.createElement("td", null, "Esther"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-development-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-development-input" }, "Desarrollo"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "4"),
                        react_1.default.createElement("td", null, "Jos\u00E9"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-validation-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-validation-input" }, "Validacion"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))))))));
};
exports.NoBorder.story = {
    name: 'Sin separadores'
};
exports.NormalTable = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-table-responsive" },
        react_1.default.createElement("div", { className: "responsive-scroll", tabIndex: 0 },
            react_1.default.createElement("table", { className: "table" },
                react_1.default.createElement("thead", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("th", { scope: "col", className: "tb-number" }, "N\u00FAmero"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-link" }, "Enlace"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-button" }, "Bot\u00F3n"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-check" }, "Checkbox"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-tag" }, "Etiqueta"))),
                react_1.default.createElement("tbody", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "1"),
                        react_1.default.createElement("td", null, "Juan"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" }, "Redacci\u00F3n"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "2"),
                        react_1.default.createElement("td", null, "Mar\u00EDa"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-design-input", name: "skills", value: "design" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-design-input" }, "Dise\u00F1o"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "3"),
                        react_1.default.createElement("td", null, "Esther"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-development-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-development-input" }, "Desarrollo"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "4"),
                        react_1.default.createElement("td", null, "Jos\u00E9"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-validation-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-validation-input" }, "Validacion"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))))))));
};
exports.NormalTable.story = {
    name: 'Con separadores'
};
exports.StripedTable = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-table-responsive" },
        react_1.default.createElement("div", { className: "responsive-scroll", tabIndex: 0 },
            react_1.default.createElement("table", { className: "table table-striped" },
                react_1.default.createElement("thead", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("th", { scope: "col", className: "tb-number" }, "N\u00FAmero"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-link" }, "Enlace"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-button" }, "Bot\u00F3n"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-check" }, "Checkbox"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-tag" }, "Etiqueta"))),
                react_1.default.createElement("tbody", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "1"),
                        react_1.default.createElement("td", null, "Juan"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" }, "Redacci\u00F3n"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "2"),
                        react_1.default.createElement("td", null, "Mar\u00EDa"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-design-input", name: "skills", value: "design" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-design-input" }, "Dise\u00F1o"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "3"),
                        react_1.default.createElement("td", null, "Esther"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-development-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-development-input" }, "Desarrollo"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", { className: "text-right" }, "4"),
                        react_1.default.createElement("td", null, "Jos\u00E9"),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "Enlace predeterminado")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("button", { type: "button", className: "btn btn-primary btn-icon" },
                                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                                "bot\u00F3n")),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-validation-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-validation-input" }, "Validacion"))),
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("span", { className: "badge badge-secondary" }, "Etiqueta 1"))))))));
};
exports.StripedTable.story = {
    name: 'Con patrón cebra'
};
exports.CheckboxAndText = function () {
    var _a = react_1.default.useState(true), value = _a[0], setValue = _a[1];
    return (react_1.default.createElement("div", { className: "storybook__container-table-responsive" },
        react_1.default.createElement("div", { className: "responsive-scroll", tabIndex: 0 },
            react_1.default.createElement("table", { className: "table" },
                react_1.default.createElement("thead", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("th", { scope: "col", className: "tb-check" }, "Checkbox"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-number" }, "N\u00FAmero"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"),
                        react_1.default.createElement("th", { scope: "col", className: "tb-text" }, "Texto"))),
                react_1.default.createElement("tbody", null,
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite", checked: value, onChange: function (event) { return setValue(event.target.checked); } }),
                                react_1.default.createElement("label", { className: "custom-control-label no-text", htmlFor: "skills-copywrite-input", "aria-label": "Descripci\u00F3n para usuarios de lectores de pantalla" }))),
                        react_1.default.createElement("td", { className: "text-right" }, "$100,00"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado")),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-design-input", name: "skills", value: "design" }),
                                react_1.default.createElement("label", { className: "custom-control-label no-text", htmlFor: "skills-design-input", "aria-label": "Descripci\u00F3n para usuarios de lectores de pantalla" }))),
                        react_1.default.createElement("td", { className: "text-right" }, "$100,00"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado")),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-development-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label no-text", htmlFor: "skills-development-input", "aria-label": "Descripci\u00F3n para usuarios de lectores de pantalla" }))),
                        react_1.default.createElement("td", { className: "text-right" }, "$100,00"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado")),
                    react_1.default.createElement("tr", null,
                        react_1.default.createElement("td", null,
                            react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
                                react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-validation-input", name: "skills", value: "development" }),
                                react_1.default.createElement("label", { className: "custom-control-label no-text", htmlFor: "skills-validation-input", "aria-label": "Descripci\u00F3n para usuarios de lectores de pantalla" }))),
                        react_1.default.createElement("td", { className: "text-right" }, "$100,00"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado"),
                        react_1.default.createElement("td", null, "Texto predeterminado")))))));
};
exports.CheckboxAndText.story = {
    name: 'Con checkbox'
};
