// @gcba/obelisco/dist/organisms/Table/TableActions.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ConEnlaces = exports.ConBotones = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var Button_1 = require("../../components/Button");
// Configuración general del componente
exports.default = {
    title: 'Organismos|Tablas/Con acciones',
    component: _1.Table,
    decorators: [addon_a11y_1.withA11y]
};
exports.ConBotones = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', 'Otro más'], body: [
            ['1', 'Juan', ActionButton],
            ['2', 'María', ActionButton],
            ['3', 'Esther', ActionButton],
            ['4', 'José', ActionButton]
        ] }));
};
exports.ConBotones.story = {
    name: 'Con botones'
};
exports.ConEnlaces = function () {
    return (react_1.default.createElement(_1.Table, { head: ['#', 'Primero', 'Otro más'], body: [
            ['1', 'Juan', ActionLinks],
            ['2', 'María', ActionLinks],
            ['3', 'Esther', ActionLinks],
            ['4', 'José', ActionLinks]
        ] }));
};
exports.ConEnlaces.story = {
    name: 'Con enlaces'
};
// Utils
var ActionButton = (react_1.default.createElement(Button_1.Button, { type: "success", size: "small" }, "Acci\u00F3n"));
var ActionLinks = (react_1.default.createElement("a", { className: "", href: "#", target: "_blank", rel: "noreferrer" }, "Enlace"));
