// @gcba/obelisco/dist/components/Card/CardsList.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnaColumna = exports.DosColumnas = exports.TresColumnas = exports.CuatroColumnas = void 0;
// Base
var react_1 = require("react");
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Tarjetas/Lista ancho completo',
    component: _1.Card,
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de tarjeta';
var descriptionSm = 'Recomendamos en este espacio no utilizar más de 3 líneas.';
var description = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var descriptionLg = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima. En este caso agregamos líneas de texto extras y mostrar el comportamiento de la card.';
exports.CuatroColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }))));
};
exports.CuatroColumnas.story = { name: 'Cuatro columnas' };
exports.TresColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-3" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }))));
};
exports.TresColumnas.story = { name: 'Tres columnas' };
exports.DosColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-2" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }))));
};
exports.DosColumnas.story = { name: 'Dos columnas' };
exports.UnaColumna = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-1" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }))));
};
exports.UnaColumna.story = { name: 'Una columna' };
