// @gcba/obelisco/dist/components/Card/Card.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SimpleEventCard = exports.TitleIconTextLink = exports.TitleTextLink = exports.HorizontalCard = exports.TitleSubtitleLink = exports.IconWithBackGround = exports.Icon = exports.Tags = exports.TitleSubtitleText = void 0;
// Base
var react_1 = require("react");
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Tarjetas/Tipos',
    component: _1.Card,
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de tarjeta';
var subTitle = 'Subtítulo de tarjeta';
var description = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var footer = 'Last updated 3 mins ago';
exports.TitleSubtitleText = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: title, subTitle: subTitle, description: description, footer: footer, href: "#" })));
};
exports.TitleSubtitleText.story = { name: 'Contenedor simple' };
exports.Tags = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: title, description: description, href: "#", tags: ['Etiqueta 1', 'Etiqueta 2'] })));
};
exports.Tags.story = { name: 'Con etiquetas' };
exports.Icon = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                icon: 'bx-news',
                color: '#009afd'
            }, href: "#" })));
};
exports.Icon.story = { name: 'Con ícono' };
exports.IconWithBackGround = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("div", { className: "cards-container" },
            react_1.default.createElement("a", { className: "card", href: "#" },
                react_1.default.createElement("div", { className: "card-body" },
                    react_1.default.createElement("i", { className: "bx bx-news card-icon card-icon-with-bg" }),
                    react_1.default.createElement("h3", { className: "card-title" }, "T\u00EDtulo de tarjeta"),
                    react_1.default.createElement("p", { className: "card-text" }, "Recomendamos en este espacio no utilizar m\u00E1s de 3 l\u00EDneas, para generar una lectura \u00F3ptima."))))));
};
exports.IconWithBackGround.story = { name: 'Ícono con fondo' };
exports.TitleSubtitleLink = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { tags: ['Etiqueta 1', 'Etiqueta 2'], title: title, subTitle: subTitle, description: description, picture: {
                src: 'cards/paseobajo.jpg',
                alt: 'descripción alternativa'
            }, href: "#" })));
};
exports.TitleSubtitleLink.story = { name: 'Contenedor con imagen' };
exports.HorizontalCard = function () {
    return (react_1.default.createElement("div", { className: "horizontal-container" },
        react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                src: 'cards/paseobajo-wide.png',
                alt: 'descripción alternativa',
                width: 168
            }, orientation: "horizontal", href: "#" })));
};
exports.HorizontalCard.story = { name: 'Contenedor con imagen horizontal' };
exports.TitleTextLink = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.SimpleCard, { title: title, description: description, href: "#" })));
};
exports.TitleTextLink.story = { name: 'Sin contenedor' };
exports.TitleIconTextLink = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.SimpleCard, { title: title, description: description, href: "#", picture: {
                icon: 'bx-news',
                color: '#26874a'
            } })));
};
exports.TitleIconTextLink.story = { name: 'Sin contenedor con ícono' };
exports.SimpleEventCard = function () {
    var eventTitle = 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed';
    var eventSubtitle = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
    var footer = (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("i", { className: "bx bx-calendar-event" }),
        " Vie 21/1",
        react_1.default.createElement("br", null),
        react_1.default.createElement("i", { className: "bx bx-time-five" }),
        " 17 | 18 | 19 hs"));
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: eventTitle, description: eventSubtitle, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" })));
};
exports.SimpleEventCard.story = { name: 'Evento simple' };
