// @gcba/obelisco/dist/components/Card/Card.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.EventCard = exports.SimpleEventCard = exports.NewsCard = exports.HorizontalCard = exports.TitleText = exports.TitleTextLink = void 0;
// Base
var react_1 = require("react");
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Tarjetas/Tipos',
    component: _1.Card,
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de tarjeta';
var description = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var footer = 'Fecha de publicación: 10/10/2021';
exports.TitleTextLink = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "horizontal-container" },
            react_1.default.createElement(_1.SimpleCard, { title: title, description: description, picture: {
                    src: 'cards/ciudad.jpg',
                    alt: 'descripción alternativa'
                }, className: "card-lg", href: "#" })),
        react_1.default.createElement("br", null),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "cards-container" },
            react_1.default.createElement(_1.SimpleCard, { title: title, description: description, href: "#", picture: {
                    icon: 'bx-news'
                } }),
            react_1.default.createElement("br", null),
            react_1.default.createElement("br", null),
            react_1.default.createElement(_1.SimpleCard, { title: "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed", description: "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna", href: "#" }),
            react_1.default.createElement("br", null),
            react_1.default.createElement("br", null),
            react_1.default.createElement(_1.SimpleCard, { title: "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed", href: "#" }))));
};
exports.TitleTextLink.story = { name: 'Sin contenedor' };
exports.TitleText = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                icon: 'bx-news'
            }, href: "#" }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.Card, { title: "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed", href: "#" })));
};
exports.TitleText.story = { name: 'Con contenedor' };
exports.HorizontalCard = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "horizontal-container" },
            react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                    src: 'cards/paseobajo-wide.png',
                    alt: 'descripción alternativa'
                }, orientation: "horizontal", href: "#" }),
            react_1.default.createElement("br", null),
            react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                    src: 'cards/paseobajo-wide.png',
                    alt: 'descripción alternativa'
                }, orientation: "horizontal", className: "card-lg", href: "#" })),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "cards-container" },
            react_1.default.createElement(_1.Card, { title: "Lorem ipsum dolor sit amet, consetetur", picture: {
                    src: 'cards/paseobajo-wide.png',
                    alt: 'descripción alternativa'
                }, orientation: "horizontal", href: "#" }),
            react_1.default.createElement("br", null),
            react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                    src: 'cards/paseobajo-wide.png',
                    alt: 'descripción alternativa'
                }, orientation: "horizontal", href: "#" }),
            react_1.default.createElement("br", null),
            react_1.default.createElement(_1.Card, { title: title, description: description, picture: {
                    src: 'cards/paseobajo-wide.png',
                    alt: 'descripción alternativa'
                }, orientation: "horizontal", className: "card-lg", href: "#" }))));
};
exports.HorizontalCard.story = { name: 'Contenedor con imagen horizontal' };
exports.NewsCard = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { tags: ['Etiqueta 1', 'Etiqueta 2'], title: title, description: description, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" })));
};
exports.NewsCard.story = { name: 'Noticia' };
exports.SimpleEventCard = function () {
    var eventTitle = 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed';
    var eventSubtitle = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
    var footer = (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("i", { className: "bx bx-calendar-event" }),
        " Vie 21/1",
        react_1.default.createElement("br", null),
        react_1.default.createElement("i", { className: "bx bx-time-five" }),
        " 17 | 18 | 19 hs"));
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement(_1.Card, { title: eventTitle, description: eventSubtitle, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" })));
};
exports.SimpleEventCard.story = { name: 'Evento simple' };
exports.EventCard = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("div", { className: "card-event-top" },
                react_1.default.createElement("img", { src: "cards/evento-bg.jpg", className: "card-event-bg", alt: "descripci\u00F3n alternativa" }),
                react_1.default.createElement("div", { className: "card-event-img" },
                    react_1.default.createElement("img", { src: "cards/evento-thumb.jpg", alt: "descripci\u00F3n alternativa" })),
                react_1.default.createElement("p", { className: "card-event-name" }, "Actividades y clases en polideportivos")),
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("p", { className: "card-headline" }, "18 de Enero al 1 de Febrero"),
                react_1.default.createElement("h4", { className: "card-title" }, "Comienza una nueva edici\u00F3n de actividades en polideportivos")))));
};
exports.EventCard.story = { name: 'Evento' };
