// @gcba/obelisco/dist/components/Card/CardsHorizontal.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Tematic = exports.OnlyText = exports.WithImage = exports.WithIcon = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Tarjetas/Horizontal',
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de la tarjeta con más de una línea';
var description = 'Descripción de la tarjeta';
exports.WithIcon = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-card-horizontal" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", icon: "bx bxs-info-circle", isHorizontal: true }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", icon: "info", isHorizontal: true, hasNoBorder: true })));
};
exports.WithIcon.story = { name: 'Con ícono' };
exports.WithImage = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-card-horizontal" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/img-left.jpg', alt: 'descripción de imagen' }, isHorizontal: true }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/img-left.jpg', alt: 'descripción de imagen' }, isHorizontal: true, hasNoBorder: true })));
};
exports.WithImage.story = { name: 'Con imagen' };
exports.OnlyText = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-card-horizontal" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", isHorizontal: true }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", isHorizontal: true, hasNoBorder: true })));
};
exports.OnlyText.story = { name: 'Solo texto' };
exports.Tematic = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-card-horizontal" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/home.svg', alt: 'descripción de imagen' }, isThemeCard: true, hasNoBorder: true })));
};
exports.Tematic.story = { name: 'Temática' };
