// @gcba/obelisco/dist/components/Card/CardListScroll.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ScrollList = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Tarjetas/Lista desplazable',
    component: _1.Card,
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de tarjeta';
var description = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var footer = 'Fecha de publicación: 10/09/2023';
exports.ScrollList = function () {
    return (react_1.default.createElement("div", { className: "responsive-scroll vertical-cards-container", tabIndex: 0 },
        react_1.default.createElement(_1.Card, { tags: ['Etiqueta 1', 'Etiqueta 2'], title: title, description: description, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" }),
        react_1.default.createElement(_1.Card, { tags: ['Etiqueta 1', 'Etiqueta 2'], title: title, description: description, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" }),
        react_1.default.createElement(_1.Card, { tags: ['Etiqueta 1', 'Etiqueta 2'], title: title, description: description, footer: footer, picture: {
                src: 'cards/evento.jpg',
                alt: 'descripción alternativa'
            }, href: "#" })));
};
exports.ScrollList.story = { name: 'Lista desplazable' };
