// @gcba/obelisco/dist/components/Card/CardsList.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnaColumna = exports.DosColumnas = exports.TresColumnas = exports.CuatroColumnas = void 0;
// Base
var react_1 = require("react");
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Tarjetas/Listas',
    component: _1.Card,
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de tarjeta';
var descriptionSm = 'Lorem ipsum dolor sit amet consectetur adipisicing elit.';
var description = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var descriptionLg = 'Rerum officiis ullam, dolore quisquam nisi eveniet ratione possimus, fugit iure saepe praesentium aliquam est facilis quibusdam nobis velit dolorum vel nesciunt.';
exports.CuatroColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }))));
};
exports.TresColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-3" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }))));
};
exports.DosColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-2" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }))));
};
exports.UnaColumna = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "card-deck max-cards-1" },
            react_1.default.createElement(_1.Card, { title: title, description: descriptionSm, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: description, href: "#" }),
            react_1.default.createElement(_1.Card, { title: title, description: descriptionLg, href: "#" }))));
};
