// @gcba/obelisco/dist/components/Card/CardsVertical.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Event = exports.News = exports.WithImage = exports.WithIcon = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Card.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Tarjetas/Vertical',
    decorators: [addon_a11y_1.withA11y]
};
var title = 'Título de la tarjeta';
var titleExtended = 'Título de la tarjeta Noticias truncado con texto extendido que supera las tres líneas de longitud.';
var description = 'Descripción de la tarjeta';
exports.WithIcon = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-cards" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", icon: "bx bxs-info-circle" }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", icon: "info", hasNoBorder: true })));
};
exports.WithIcon.story = { name: 'Con ícono' };
exports.WithImage = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-cards" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/img-top.jpg', alt: 'descripción de imagen' } })));
};
exports.WithImage.story = { name: 'Con imagen' };
exports.News = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-cards" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/img-top.jpg', alt: 'descripción de imagen' }, tags: [{ text: 'etiqueta 1' }, { text: 'etiqueta 2' }] },
            react_1.default.createElement("small", null, "Fecha de publicaci\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.CardComponent, { title: titleExtended, description: description, href: "#", image: { src: 'cards/img-top.jpg', alt: 'descripción de imagen' }, tags: [{ text: 'etiqueta 1' }, { text: 'etiqueta 2' }], isTitleTruncate: true },
            react_1.default.createElement("small", null, "Fecha de publicaci\u00F3n"))));
};
exports.News.story = { name: 'Noticia' };
exports.Event = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-cards" },
        react_1.default.createElement(_1.CardComponent, { title: title, description: description, href: "#", image: { src: 'cards/img-top.jpg', alt: 'descripción de imagen' }, headline: "sobrelinea" },
            react_1.default.createElement("div", { className: "pt-2" },
                react_1.default.createElement("small", null,
                    react_1.default.createElement("span", { className: "material-icons-round" }, "calendar_today"),
                    " Lun 01/1"),
                react_1.default.createElement("small", null,
                    react_1.default.createElement("span", { className: "material-icons-round" }, "watch_later"),
                    " 17 | 18 | 19 hs")))));
};
exports.Event.story = { name: 'Agenda' };
