// @gcba/obelisco/dist/components/Alert/Alert.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Lista = exports.Colores = exports.Titulo = exports.Texto = void 0;
// Base
var react_1 = require("react");
require("./Alert.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var utils_1 = require("../utils");
// Config
exports.default = {
    title: 'Componentes|Alerta',
    decorators: [addon_a11y_1.withA11y]
};
exports.Texto = function () {
    return (react_1.default.createElement("div", { className: "alert-wrapper" },
        react_1.default.createElement("div", { className: "alert alert-danger", role: "alert" },
            react_1.default.createElement("p", null,
                "Un texto simple para una alerta simple, incluso\u00A0",
                react_1.default.createElement("a", { href: "#" }, "con hiperv\u00EDnculos"),
                "."))));
};
exports.Titulo = function () {
    return (react_1.default.createElement("div", { className: "alert-wrapper" },
        react_1.default.createElement("div", { className: "alert alert-danger", role: "alert" },
            react_1.default.createElement("p", null,
                react_1.default.createElement("strong", null, "Este es un destacado de una alerta (opcional). "),
                "Un texto simple para una alerta simple, incluso\u00A0",
                react_1.default.createElement("a", { href: "#" }, "con hiperv\u00EDnculos"),
                "."))));
};
exports.Titulo.story = {
    name: 'Destacado y texto'
};
exports.Colores = function () {
    return (react_1.default.createElement("div", { className: "alert-wrapper" }, utils_1.alertTypes.map(function (type) { return (react_1.default.createElement("div", { key: type, className: "alert alert-" + type, role: "alert" },
        react_1.default.createElement("p", null, "Un texto simple para una alerta simple."))); })));
};
exports.Lista = function () {
    return (react_1.default.createElement("div", { className: "alert-wrapper" },
        react_1.default.createElement("div", { className: "alert alert-danger fade show", role: "alert" },
            react_1.default.createElement("p", null,
                react_1.default.createElement("strong", null, "Este es un destacado de una alerta (opcional). ")),
            react_1.default.createElement("ol", null,
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { href: "#" }, "Un texto simple para un enlace en una lista de alerta")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { href: "#" }, "Un texto simple para un enlace en una lista de alerta")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { href: "#" }, "Un texto simple para un enlace en una lista de alerta"))))));
};
