// @gcba/obelisco/dist/components/Pagination/PaginationNumbered.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Final = exports.Central = exports.Initial = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Paginado/Numerado',
    component: _1.Pagination,
    decorators: [addon_a11y_1.withA11y]
};
exports.Initial = function () { return (react_1.default.createElement("div", { className: "storybook__container-pagination" },
    react_1.default.createElement(_1.Pagination, { pages: 10, ariaLabel: "Ejemplo de paginado numerado inicial" }))); };
exports.Initial.story = { name: 'Inicial' };
exports.Central = function () { return (react_1.default.createElement("div", { className: "storybook__container-pagination" },
    react_1.default.createElement(_1.Pagination, { pages: 10, default: 4, ariaLabel: "Ejemplo de paginado numerado central" }))); };
exports.Final = function () { return (react_1.default.createElement("div", { className: "storybook__container-pagination" },
    react_1.default.createElement(_1.Pagination, { pages: 10, default: 9, ariaLabel: "Ejemplo de paginado numerado final" }))); };
