// @gcba/obelisco/dist/components/List-group/Horizontal/HorizontalAccess.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SmallHorizontalAccessWithIcon = exports.HorizontalAccessWithIcon = exports.SmallHorizontalAccessDescription = exports.HorizontalAccessSmall = exports.HorizontalAccess = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Accesos/Horizontal',
    decorators: [addon_a11y_1.withA11y]
};
exports.HorizontalAccess = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "list-group list-group-row" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item hover" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item focus" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item list-group-item-logout" },
            react_1.default.createElement("span", { className: "access-title" }, "Cerrar sesi\u00F3n "))))); };
exports.HorizontalAccess.story = { name: 'Acceso horizontal' };
exports.HorizontalAccessSmall = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "list-group list-group-row" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm " },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm hover" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm focus" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm list-group-item-logout" },
            react_1.default.createElement("span", { className: "access-title" }, "Cerrar sesi\u00F3n "))))); };
exports.HorizontalAccessSmall.story = {
    name: 'Acceso horizontal chico'
};
exports.SmallHorizontalAccessDescription = function () { return (react_1.default.createElement("div", { className: "list-group list-group-row" },
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
        react_1.default.createElement("div", { className: "access-content" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
            react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n"))),
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm hover" },
        react_1.default.createElement("div", { className: "access-content" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
            react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n"))),
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm focus" },
        react_1.default.createElement("div", { className: "access-content" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
            react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n"))))); };
exports.SmallHorizontalAccessDescription.story = {
    name: 'Acceso horizontal chico con descripción'
};
exports.HorizontalAccessWithIcon = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "list-group list-group-row" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-facebook-circle" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-instagram" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-twitter" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-youtube" }),
                "Acceso grande"))),
    react_1.default.createElement("div", { className: "card-deck max-cards-2" },
        react_1.default.createElement("div", { className: "group-access", style: { width: '324px' } },
            react_1.default.createElement("div", { className: "card-access" },
                react_1.default.createElement("div", { className: "card-access-body reverse" },
                    react_1.default.createElement("div", null,
                        react_1.default.createElement("h3", { className: "card-access-title" }, "T\u00EDtulo"),
                        react_1.default.createElement("p", { className: "card-access-text" }, "Bajada")),
                    react_1.default.createElement("i", { className: "bx bxs-shield-x card-access-icon" })),
                react_1.default.createElement("a", { href: "#", className: "list-group-item card-item" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")))),
        react_1.default.createElement("div", { className: "group-access", style: { width: '324px' } },
            react_1.default.createElement("div", { className: "card-access" },
                react_1.default.createElement("div", { className: "card-access-body reverse" },
                    react_1.default.createElement("div", null,
                        react_1.default.createElement("h3", { className: "card-access-title" }, "T\u00EDtulo"),
                        react_1.default.createElement("p", { className: "card-access-text" }, "Bajada")),
                    react_1.default.createElement("i", { className: "bx bxs-shield-x card-access-icon" })),
                react_1.default.createElement("a", { href: "#", className: "list-group-item card-item" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")))),
        react_1.default.createElement("div", { className: "group-access", style: { width: '324px' } },
            react_1.default.createElement("div", { className: "card-access" },
                react_1.default.createElement("div", { className: "card-access-body reverse" },
                    react_1.default.createElement("div", null,
                        react_1.default.createElement("h3", { className: "card-access-title" }, "T\u00EDtulo"),
                        react_1.default.createElement("p", { className: "card-access-text" }, "Bajada")),
                    react_1.default.createElement("i", { className: "bx bxs-shield-x card-access-icon" })),
                react_1.default.createElement("a", { href: "#", className: "list-group-item card-item" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso grande"))))))); };
exports.HorizontalAccessWithIcon.story = {
    name: 'Acceso horizontal con ícono'
};
exports.SmallHorizontalAccessWithIcon = function () { return (react_1.default.createElement("div", { className: "list-group list-group-row" },
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
        react_1.default.createElement("span", { className: "access-title" },
            react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-facebook-circle" }),
            "Acceso chico")),
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm " },
        react_1.default.createElement("span", { className: "access-title" },
            react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-instagram" }),
            "Acceso chico")),
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm " },
        react_1.default.createElement("span", { className: "access-title" },
            react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-twitter" }),
            "Acceso chico")),
    react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm " },
        react_1.default.createElement("span", { className: "access-title" },
            react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-youtube" }),
            "Acceso chico")))); };
exports.SmallHorizontalAccessWithIcon.story = {
    name: 'Acceso horizontal chico con ícono'
};
