// @gcba/obelisco/dist/components/Gallery/GalleryGrid.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnaImagen = exports.DosImagenes = exports.TresImagenes = exports.CuatroImagenes = exports.CincoImagenes = exports.SeisImagenes = exports.SieteImagenes = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Gallery.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var index_1 = require("./index");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Galería/Grillas',
    decorators: [addon_a11y_1.withA11y]
};
// Función para crear los componentes
var createGridComponent = function (name, total) {
    var Component = function () { return (react_1.default.createElement("div", { className: "storybook__container-grid" },
        react_1.default.createElement(index_1.GridImage, { total: total }))); };
    Component.story = {
        name: name
    };
    return Component;
};
// Creación de los componentes utilizando la función
exports.SieteImagenes = createGridComponent('Siete imágenes', 7);
exports.SeisImagenes = createGridComponent('Seis imágenes', 6);
exports.CincoImagenes = createGridComponent('Cinco imágenes', 5);
exports.CuatroImagenes = createGridComponent('Cuatro imágenes', 4);
exports.TresImagenes = createGridComponent('Tres imágenes', 3);
exports.DosImagenes = createGridComponent('Dos imágenes', 2);
exports.UnaImagen = createGridComponent('Una imagen', 1);
