// @gcba/obelisco/dist/components/Gallery/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.GalleryContainer = exports.GridImage = exports.images = void 0;
var classnames_1 = __importDefault(require("classnames"));
var react_1 = __importDefault(require("react"));
function createImageObject(number) {
    return {
        url: "gallery/" + number + ".jpg",
        alt: 'Texto alternativo de la imagen',
        author: 'Autor de la imagen.',
        title: 'Título de la imagen',
        description: 'Descripción o epígrafe de la imagen. \nRecomendamos en este espacio no utilizar más de 2 líneas, para generar una lectura óptima.'
    };
}
exports.images = [
    createImageObject(1),
    createImageObject(2),
    createImageObject(3),
    createImageObject(4),
    createImageObject(5),
    createImageObject(3),
    createImageObject(4)
];
exports.GridImage = function (props) {
    var _a = props.total, total = _a === void 0 ? exports.images.length : _a, _b = props.interactive, interactive = _b === void 0 ? false : _b;
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "gallery-grid gallery-max-" + total }, exports.images.slice(0, total).map(function (_a, index) {
            var url = _a.url, alt = _a.alt;
            return interactive ? (react_1.default.createElement("a", { key: index, href: url, className: "gallery-grid-item", "data-toggle": "modal", "data-target": "#modalGallery" },
                react_1.default.createElement("img", { src: url, alt: alt }))) : (react_1.default.createElement("div", { key: index, className: "gallery-grid-item" },
                react_1.default.createElement("img", { src: url, alt: alt })));
        }))));
};
exports.GalleryContainer = function (props) {
    var _a = props.total, total = _a === void 0 ? exports.images.length : _a, _b = props.show, show = _b === void 0 ? false : _b;
    var modalClass = classnames_1.default('modal', 'modal-carousel', { 'd-block': show });
    return (react_1.default.createElement("div", { className: modalClass, "data-backdrop": "static", tabIndex: -1, role: "dialog", id: "modalGallery" },
        react_1.default.createElement("div", { className: "modal-dialog modal-lg", role: "document" },
            react_1.default.createElement("div", { className: "modal-content" },
                react_1.default.createElement("a", { href: "#", className: "modal-carousel-close", "data-dismiss": "modal" }, "Cerrar ventana"),
                react_1.default.createElement("div", { id: "modalGalleryControls", className: "carousel", "data-ride": "carousel", "data-interval": "false" },
                    react_1.default.createElement("div", { className: "carousel-inner" }, exports.images.slice(0, total).map(function (_a, index) {
                        var url = _a.url, alt = _a.alt, author = _a.author, title = _a.title, description = _a.description;
                        return (react_1.default.createElement("div", { key: index, className: "carousel-item " + (index === 0 ? 'active' : '') },
                            react_1.default.createElement("img", { src: url, className: "d-block w-100", alt: alt }),
                            react_1.default.createElement("div", { className: "carousel-caption" },
                                react_1.default.createElement("div", { className: "row mb-4" },
                                    react_1.default.createElement("div", { className: "col" },
                                        react_1.default.createElement("span", { className: "text-xs" }, author)),
                                    react_1.default.createElement("div", { className: "col text-right" },
                                        react_1.default.createElement("span", { className: "text-xs" },
                                            "Imagen ",
                                            index + 1,
                                            "/",
                                            total))),
                                react_1.default.createElement("h5", { className: "h2" },
                                    title,
                                    " ",
                                    index + 1,
                                    "."),
                                description && (react_1.default.createElement("p", null, description.split('\n').map(function (line, index) { return (react_1.default.createElement(react_1.default.Fragment, { key: index },
                                    index === 0 ? line + " " : line,
                                    index === 0 && react_1.default.createElement("br", null))); }))))));
                    })),
                    react_1.default.createElement("button", { className: "carousel-control-prev", type: "button", "data-target": "#modalGalleryControls", "data-slide": "prev" },
                        react_1.default.createElement("span", { className: "carousel-control-prev-icon", "aria-hidden": "true" }),
                        react_1.default.createElement("span", { className: "sr-only" }, "Anterior")),
                    react_1.default.createElement("button", { className: "carousel-control-next", type: "button", "data-target": "#modalGalleryControls", "data-slide": "next" },
                        react_1.default.createElement("span", { className: "carousel-control-next-icon", "aria-hidden": "true" }),
                        react_1.default.createElement("span", { className: "sr-only" }, "Siguiente")))))));
};
