// @gcba/obelisco/dist/form/Checkbox.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.BotonCheck = exports.Deshabilitado = exports.TerminosYCondiciones = exports.CasillaMultiple = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Formulario|Checkbox',
    decorators: [addon_a11y_1.withA11y]
};
exports.CasillaMultiple = function () {
    var _a = react_1.default.useState(true), value = _a[0], setValue = _a[1];
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", null, "Seleccion\u00E1 una o varias opciones *"),
            react_1.default.createElement("span", { className: "form-text" }, "Texto de asistencia")),
        react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite", checked: value, onChange: function (event) { return setValue(event.target.checked); } }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" }, "Redacci\u00F3n")),
        react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-design-input", name: "skills", value: "design" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-design-input" }, "Dise\u00F1o")),
        react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-development-input", name: "skills", value: "development" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-development-input" }, "Desarrollo")),
        react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
            react_1.default.createElement("input", { className: "custom-control-input is-invalid", type: "checkbox", id: "skills-validation-input", name: "skills", value: "development" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-validation-input" }, "Validacion"))));
};
exports.CasillaMultiple.story = {
    name: 'Casilla múltiple'
};
exports.TerminosYCondiciones = function () {
    return (react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
        react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite" }),
        react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" },
            "Acepto los ",
            react_1.default.createElement("a", { href: "#" }, "t\u00E9rminos y condiciones"))));
};
exports.TerminosYCondiciones.story = {
    name: 'Términos y condiciones'
};
exports.Deshabilitado = function () {
    return (react_1.default.createElement("div", { className: "custom-control custom-checkbox" },
        react_1.default.createElement("input", { className: "custom-control-input", type: "checkbox", id: "skills-copywrite-input", name: "skills", value: "copywrite", disabled: true }),
        react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "skills-copywrite-input" }, "Deshabilitado")));
};
exports.BotonCheck = function () {
    return (react_1.default.createElement("div", { className: "btn-wrapper" },
        react_1.default.createElement("input", { type: "checkbox", className: "btn-check", id: "btn-check", autoComplete: "off" }),
        react_1.default.createElement("label", { className: "btn btn-chip btn-sm", htmlFor: "btn-check" }, "Bot\u00F3n"),
        react_1.default.createElement("input", { type: "checkbox", className: "btn-check", id: "btn-check2", autoComplete: "off" }),
        react_1.default.createElement("label", { className: "btn btn-chip btn-sm", htmlFor: "btn-check2" }, "Bot\u00F3n"),
        react_1.default.createElement("input", { type: "checkbox", className: "btn-check", id: "btn-check3", autoComplete: "off", disabled: true }),
        react_1.default.createElement("label", { className: "btn btn-chip btn-sm disabled", htmlFor: "btn-check3" }, "Bot\u00F3n")));
};
exports.BotonCheck.story = { name: 'Botón de selección mútiple' };
