// @gcba/obelisco/dist/form/Radio.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.BotonRadio = exports.Deshabilitado = exports.CasillaUnica = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Formulario|Radio',
    decorators: [addon_a11y_1.withA11y]
};
exports.CasillaUnica = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", null, "Seleccion\u00E1 una opci\u00F3n *"),
            react_1.default.createElement("span", { className: "form-text" }, "Texto de asistencia")),
        react_1.default.createElement("div", { className: "custom-control custom-radio" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "radio", name: "profession", id: "profession-designer", value: "designer", defaultChecked: true }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "profession-designer" }, "Dise\u00F1ador")),
        react_1.default.createElement("div", { className: "custom-control custom-radio" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "radio", name: "profession", id: "profession-developer", value: "developer" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "profession-developer" }, "Desarrollador")),
        react_1.default.createElement("div", { className: "custom-control custom-radio" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "radio", name: "profession", id: "profession-content", value: "content" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "profession-content" }, "Redactor de contenidos")),
        react_1.default.createElement("div", { className: "custom-control custom-radio" },
            react_1.default.createElement("input", { className: "custom-control-input is-invalid", type: "radio", name: "profession", id: "validation-content", value: "content" }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "validation-content" }, "Validaci\u00F3n"))));
};
exports.CasillaUnica.story = {
    name: 'Casilla única'
};
exports.Deshabilitado = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "custom-control custom-radio" },
            react_1.default.createElement("input", { className: "custom-control-input", type: "radio", name: "profession", id: "profession-designer", value: "designer", disabled: true }),
            react_1.default.createElement("label", { className: "custom-control-label", htmlFor: "profession-designer" }, "Deshabilitado"))));
};
exports.BotonRadio = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "btn-wrapper", "data-toggle": "buttons" },
            react_1.default.createElement("label", { className: "btn btn-chip btn-sm" },
                react_1.default.createElement("input", { type: "radio", name: "options1", id: "option1", className: "btn-check" }),
                "Bot\u00F3n"),
            react_1.default.createElement("label", { className: "btn btn-chip btn-sm active" },
                react_1.default.createElement("input", { type: "radio", name: "options2", id: "option2", className: "btn-check", checked: true }),
                "Bot\u00F3n"),
            react_1.default.createElement("label", { className: "btn btn-chip btn-sm disabled " },
                react_1.default.createElement("input", { type: "radio", name: "options3", id: "option3", className: "btn-check" }),
                "Bot\u00F3n"))));
};
exports.BotonRadio.story = {
    name: 'Botón de selección única'
};
