// @gcba/obelisco/dist/form/Select.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Validacion = exports.Deshabilitado = exports.ListaDeSeleccion = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuraci贸n general del componente
exports.default = {
    title: 'Formulario|Lista de selecci贸n',
    decorators: [addon_a11y_1.withA11y]
};
exports.ListaDeSeleccion = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "lista-de-seleccion" }, "Nombre del selector *"),
            react_1.default.createElement("select", { className: "custom-select", id: "lista-de-seleccion" },
                react_1.default.createElement("option", null, "1"),
                react_1.default.createElement("option", null, "2"),
                react_1.default.createElement("option", null, "3"),
                react_1.default.createElement("option", null, "4"),
                react_1.default.createElement("option", null, "5")))));
};
exports.ListaDeSeleccion.story = { name: 'Lista de selecci贸n' };
exports.Deshabilitado = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "lista-de-seleccion" }, "Nombre del selector *"),
            react_1.default.createElement("select", { className: "custom-select", id: "lista-de-seleccion", disabled: true },
                react_1.default.createElement("option", null, "1"),
                react_1.default.createElement("option", null, "2")))));
};
exports.Validacion = function () {
    return (react_1.default.createElement("div", { className: "form-wrapper" },
        react_1.default.createElement("div", { className: "form-group" },
            react_1.default.createElement("label", { htmlFor: "lista-de-seleccion" }, "Nombre del selector *"),
            react_1.default.createElement("select", { className: "custom-select is-valid", id: "lista-de-seleccion" },
                react_1.default.createElement("option", null, "1"),
                react_1.default.createElement("option", null, "2")),
            react_1.default.createElement("div", { className: "valid-feedback" }, "La selecci\u00F3n es v\u00E1lida."),
            react_1.default.createElement("div", { className: "invalid-feedback" }, "Necesitamos que selecciones una opci\u00F3n."))));
};
exports.Validacion.story = {
    name: 'Validaci贸n'
};
