// @gcba/obelisco/dist/components/Pagination/Pagination.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.NavegacionDePagina = exports.Compleja = exports.Simple = void 0;
// Base
var react_1 = require("react");
require("./Pagination.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Paginación',
    component: _1.Pagination,
    decorators: [addon_a11y_1.withA11y]
};
exports.Simple = function () {
    return react_1.default.createElement(_1.Pagination, { pages: 3, ariaLabel: "Ejemplo de paginaci\u00F3n simple" });
};
exports.Compleja = function () {
    return react_1.default.createElement(_1.Pagination, { pages: 60, default: 9, ariaLabel: "Ejemplo de paginaci\u00F3n compleja" });
};
exports.NavegacionDePagina = function () { return (react_1.default.createElement("div", { className: "page-navigation-wrapper" },
    react_1.default.createElement("div", { className: "page-navigation" },
        react_1.default.createElement("div", { className: "row" },
            react_1.default.createElement("div", { className: "col-6" },
                react_1.default.createElement("a", { className: "page-navigation-prev", href: "#" },
                    react_1.default.createElement("span", { className: "page-previous-icon", "aria-hidden": "true" }),
                    " P\u00E1gina anterior")),
            react_1.default.createElement("div", { className: "col-6 text-right" },
                react_1.default.createElement("a", { className: "page-navigation-next", href: "#" },
                    "P\u00E1gina siguiente ",
                    react_1.default.createElement("span", { className: "page-next-icon", "aria-hidden": "true" }))))))); };
exports.NavegacionDePagina.story = {
    name: 'Navegación de página'
};
