// @gcba/obelisco/dist/components/Pagination/Pagination.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.NavegacionDePagina = exports.Paginacion = void 0;
// Base
var react_1 = require("react");
require("./Pagination.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Config
exports.default = {
    title: 'Componentes|Paginaci칩n',
    component: _1.Pagination,
    decorators: [addon_a11y_1.withA11y]
};
exports.Paginacion = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("div", { className: "mt-5" },
            react_1.default.createElement(_1.Pagination, { pages: 56, default: 0, ariaLabel: "Primer ejemplo de paginaci\u00F3n simple" })),
        react_1.default.createElement("div", { className: "mt-5" },
            react_1.default.createElement(_1.Pagination, { pages: 56, default: 1, ariaLabel: "Segundo ejemplo de paginaci\u00F3n simple" })),
        react_1.default.createElement("div", { className: "mt-5" },
            react_1.default.createElement(_1.Pagination, { pages: 56, default: 55, ariaLabel: "Tercer ejemplo de paginaci\u00F3n simple" }))));
};
exports.Paginacion.story = { name: 'Paginaci칩n' };
exports.NavegacionDePagina = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "pagination" },
        react_1.default.createElement("div", { className: "page-navigation" },
            react_1.default.createElement("div", { className: "page-item" },
                react_1.default.createElement("a", { className: "btn page-link", href: "#" },
                    react_1.default.createElement("span", { className: "page-previous-icon", "aria-hidden": "true" }),
                    react_1.default.createElement("span", { className: "page-next-text" }, "Anterior"))),
            react_1.default.createElement("div", { className: "page-item" },
                react_1.default.createElement("a", { className: "btn page-link", href: "#" },
                    react_1.default.createElement("span", { className: "page-next-text" }, "Siguiente"),
                    react_1.default.createElement("span", { className: "page-next-icon", "aria-hidden": "true" }))))))); };
exports.NavegacionDePagina.story = {
    name: 'Navegaci칩n de p치gina'
};
