// @gcba/obelisco/dist/components/Spinner/Spinner.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SpinnerBlock = exports.SpinnerSizes = exports.SpinnerTypes = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Spinner.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var utils_1 = require("../utils");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Spinner',
    component: _1.Spinner,
    decorators: [addon_a11y_1.withA11y]
};
exports.SpinnerTypes = function () { return (react_1.default.createElement("div", { className: "storybook__container-spinner" }, utils_1.spinnerTypes.map(function (type) { return (react_1.default.createElement(_1.Spinner, { key: type, type: type })); }))); };
exports.SpinnerTypes.story = {
    name: 'Tipos'
};
exports.SpinnerSizes = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-spinner" }, utils_1.sizes.map(function (item) { return (react_1.default.createElement(_1.Spinner, { key: item, type: "info", size: item })); })));
};
exports.SpinnerSizes.story = {
    name: 'Tamaños'
};
exports.SpinnerBlock = function () { return (react_1.default.createElement("div", { className: "storybook__container-spinner" },
    react_1.default.createElement(_1.Spinner, { type: "info", block: true }))); };
exports.SpinnerBlock.story = {
    name: 'Expandible'
};
