// @gcba/obelisco/dist/components/Nav/Nav-horizontal.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SimpleConBordeEIcono = exports.SimpleConBorde = exports.Tamanos = exports.SimpleConIcono = exports.Simple = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Nav.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Navegación y pestañas/Navegación horizontal',
    component: _1.NavHorizontal,
    decorators: [addon_a11y_1.withA11y]
};
var simpleItems = [
    { name: 'Link', id: '1' },
    { name: 'Activo', id: '2' },
    { name: 'Link', id: '3' },
    { name: 'Deshabilitado', id: '4', disabled: true }
];
exports.Simple = function () { return react_1.default.createElement(_1.NavHorizontal, { items: simpleItems, selected: "2" }); };
exports.SimpleConIcono = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(_1.NavHorizontal, { items: simpleItems, selected: "2", hasIcon: true }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.NavHorizontal, { items: simpleItems, selected: "2", hasIcon: true, hasReverse: true })));
};
exports.SimpleConIcono.story = {
    name: 'Simple con ícono'
};
var simpleItemsSizes = [
    { name: 'Link', id: '1', hasBordered: true, type: 'large' },
    { name: 'Link', id: '2', hasBordered: true, type: 'default' },
    { name: 'Link', id: '3', hasBordered: true, type: 'small' }
];
exports.Tamanos = function () { return react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsSizes }); };
exports.Tamanos.story = { name: 'Tamaños' };
var simpleItemsBorder = [
    { name: 'Link', id: '1', hasBordered: true },
    { name: 'Activo', id: '2', hasBordered: true },
    { name: 'Link', id: '3', hasBordered: true },
    { name: 'Deshabilitado', id: '4', disabled: true, hasBordered: true }
];
exports.SimpleConBorde = function () { return react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsBorder, selected: "2" }); };
exports.SimpleConBorde.story = {
    name: 'Simple con borde'
};
var simpleItemsBorderIcon = [
    { name: 'Link', id: '1', hasBordered: true, hasIcon: true },
    { name: 'Activo', id: '2', hasBordered: true },
    { name: 'Link', id: '3', hasBordered: true },
    { name: 'Deshabilitado', id: '4', disabled: true, hasBordered: true }
];
exports.SimpleConBordeEIcono = function () { return (react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsBorderIcon, selected: "2", hasIcon: true })); };
exports.SimpleConBordeEIcono.story = { name: 'Simple con borde e ícono' };
