// @gcba/obelisco/dist/components/Nav/Nav-horizontal.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Responsive = exports.Con3NivelesConIcono = exports.Con3Niveles = exports.Con2NivelesConIcono = exports.Con2Niveles = exports.SimpleConBordeEIcono = exports.SimpleConBorde = exports.Tamanos = exports.SimpleConIcono = exports.Simple = void 0;
// Base
var react_1 = require("react");
require("./Nav.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración del general del componente
exports.default = {
    title: 'Componentes|Navegación/Horizontal',
    component: _1.NavHorizontal,
    decorators: [addon_a11y_1.withA11y]
};
var simpleItems = [
    { name: 'Link', id: '1' },
    { name: 'Active', id: '2' },
    { name: 'Link', id: '3' },
    { name: 'Disabled', id: '4', disabled: true }
];
exports.Simple = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: simpleItems, selected: "2" }))); };
exports.SimpleConIcono = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: simpleItems, selected: "2", hasIcon: true }))); };
var simpleItemsSizes = [
    { name: 'Link', id: '1', hasBordered: true },
    { name: 'Link', id: '2', hasBordered: true, isSize: true }
];
exports.Tamanos = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsSizes }))); };
exports.Tamanos.story = { name: 'Tamaños' };
var simpleItemsBorder = [
    { name: 'Link', id: '1', hasBordered: true },
    { name: 'Active', id: '2', hasBordered: true },
    { name: 'Link', id: '3', hasBordered: true },
    { name: 'Disabled', id: '4', disabled: true, hasBordered: true }
];
exports.SimpleConBorde = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsBorder, selected: "2", hasBordered: true }))); };
var simpleItemsBorderIcon = [
    { name: 'Link', id: '1', hasBordered: true, hasIcon: true },
    { name: 'Active', id: '2', hasBordered: true },
    { name: 'Link', id: '3', hasBordered: true },
    { name: 'Disabled', id: '4', disabled: true, hasBordered: true }
];
exports.SimpleConBordeEIcono = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: simpleItemsBorderIcon, selected: "2", hasBordered: true, hasIcon: true }))); };
exports.SimpleConBordeEIcono.story = { name: 'Simple Con Borde e Icono' };
var doubleLevel = [
    { name: 'Categorías', id: '0' },
    {
        name: 'Locales y comercios',
        id: '1',
        children: [
            { name: 'Local Indumentaria y calzado. (Actividades comerciales de venta).', id: '1.1' },
            { name: 'Locales gastronómicos: “para llevar/takeaway”', id: '1.2' },
            { name: 'Locales de óptica', id: '1.3' },
            { name: 'Comercial minorista (excluyendo indumentaria y calzado)', id: '1.4' }
        ]
    },
    { name: 'Actividades jurídicas', id: '2' },
    { name: 'Otras actividades', id: '3' }
];
var tripleLevel = [
    { name: 'Categorías', id: '0' },
    {
        name: 'Locales y comercios',
        id: '1',
        children: [
            { name: 'Local Indumentaria y calzado. (Actividades comerciales de venta).', id: '1.1' },
            {
                name: 'Locales gastronómicos: “para llevar/takeaway”',
                id: '1.2',
                children: [
                    { name: 'Con local a la calle', id: '1.2.1' },
                    { name: 'Food trucks', id: '1.2.2' }
                ]
            },
            { name: 'Locales de óptica', id: '1.3' },
            { name: 'Comercial minorista (excluyendo indumentaria y calzado)', id: '1.4' }
        ]
    },
    { name: 'Actividades jurídicas', id: '2' },
    { name: 'Otras actividades', id: '3' }
];
exports.Con2Niveles = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: doubleLevel, navSize: "large", selected: "1.2" }))); };
exports.Con2NivelesConIcono = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: doubleLevel, navSize: "large", selected: "1.2", hasIcon: true }))); };
exports.Con3Niveles = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: tripleLevel, navSize: "large", selected: "1.2.2" }))); };
exports.Con3NivelesConIcono = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(_1.NavHorizontal, { items: tripleLevel, navSize: "large", selected: "1.2.2", hasIcon: true }))); };
exports.Responsive = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement("nav", null,
        react_1.default.createElement("a", { className: "nav-link nav-mobile-trigger collapsed", "data-toggle": "collapse", href: "#sidebar-nav" }, "Acerca de Obelisco"),
        react_1.default.createElement("div", { className: "nav-responsive collapse", id: "sidebar-nav" },
            react_1.default.createElement(_1.NavHorizontal, { items: tripleLevel, navSize: "large", selected: "1.2.2" }))))); };
