// @gcba/obelisco/dist/components/Nav/Nav.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Con3NivelesConIcono = exports.Con3Niveles = exports.Con2NivelesConIcono = exports.Con2Niveles = exports.SimpleConIcono = exports.Simple = void 0;
// Base
var react_1 = require("react");
require("./Nav.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Navegación/Vertical',
    component: _1.Nav,
    decorators: [addon_a11y_1.withA11y]
};
var simpleItems = [
    { name: 'Link', id: '1' },
    { name: 'Activo', id: '2' },
    { name: 'Link', id: '3' },
    { name: 'Deshabilitado', id: '4', disabled: true }
];
exports.Simple = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: simpleItems, selected: "2" }))); };
exports.SimpleConIcono = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: simpleItems, selected: "2", hasIcon: true }))); };
exports.SimpleConIcono.story = {
    name: 'Simple con ícono'
};
var doubleLevel = [
    { name: 'Categorías', id: '0' },
    {
        name: 'Locales y comercios',
        id: '1',
        children: [
            { name: 'Local Indumentaria y calzado. (Actividades comerciales de venta).', id: '1.1' },
            { name: 'Locales gastronómicos: “para llevar/takeaway”', id: '1.2' },
            { name: 'Locales de óptica', id: '1.3' },
            { name: 'Comercial minorista (excluyendo indumentaria y calzado)', id: '1.4' }
        ]
    },
    { name: 'Actividades jurídicas', id: '2' },
    { name: 'Otras actividades', id: '3' }
];
var tripleLevel = [
    { name: 'Categorías', id: '0' },
    {
        name: 'Locales y comercios',
        id: '1',
        children: [
            { name: 'Local Indumentaria y calzado. (Actividades comerciales de venta).', id: '1.1' },
            {
                name: 'Locales gastronómicos: “para llevar/takeaway”',
                id: '1.2',
                children: [
                    { name: 'Con local a la calle', id: '1.2.1' },
                    { name: 'Food trucks', id: '1.2.2' }
                ]
            },
            { name: 'Locales de óptica', id: '1.3' },
            { name: 'Comercial minorista (excluyendo indumentaria y calzado)', id: '1.4' }
        ]
    },
    { name: 'Actividades jurídicas', id: '2' },
    { name: 'Otras actividades', id: '3' }
];
exports.Con2Niveles = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: doubleLevel, navSize: "large", selected: "1.2" }))); };
exports.Con2Niveles.story = {
    name: 'Con 2 niveles'
};
exports.Con2NivelesConIcono = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: doubleLevel, navSize: "large", selected: "1.2", hasIcon: true }))); };
exports.Con2NivelesConIcono.story = {
    name: 'Con 2 niveles con ícono'
};
exports.Con3Niveles = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: tripleLevel, navSize: "large", selected: "1.2.2" }))); };
exports.Con3Niveles.story = {
    name: 'Con 3 niveles'
};
exports.Con3NivelesConIcono = function () { return (react_1.default.createElement("div", { className: "nav-container" },
    react_1.default.createElement(_1.Nav, { items: tripleLevel, navSize: "large", selected: "1.2.2", hasIcon: true }))); };
exports.Con3NivelesConIcono.story = {
    name: 'Con 3 niveles con ícono'
};
