// @gcba/obelisco/dist/components/Nav/TabsContainer.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ContainerSliderTabBg = exports.ContainerSliderTabs = exports.ContainerTabsIcon = exports.ContainerTabsBg = exports.ContainerSizeTabs = exports.ContainerTabWidth = exports.ContainerTabs = void 0;
var react_1 = __importDefault(require("react"));
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var tabs_1 = require("./tabs");
// Configuración general del componente
exports.default = {
    component: _1.Nav,
    title: 'Componentes|Navegación y pestañas/Pestañas con contenedor',
    decorators: [addon_a11y_1.withA11y]
};
var simpleItems = [
    {
        name: 'Pestaña 1',
        id: '1',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  1'
    },
    {
        name: 'Pestaña 2',
        id: '2',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  2'
    },
    {
        name: 'Pestaña 3',
        id: '3',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  3'
    },
    {
        name: 'Pestaña 4',
        id: '4',
        disabled: true,
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  4'
    }
];
var sizeLgItems = [
    {
        name: 'Pestaña grande',
        id: '5',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'large',
        content: 'Contenido Pestaña 1'
    },
    {
        name: 'Pestaña grande',
        id: '6',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'large',
        content: 'Contenido Pestaña 2'
    },
    {
        name: 'Pestaña grande',
        id: '7',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'large',
        content: 'Contenido Pestaña 3'
    }
];
var sizeItems = [
    {
        name: 'Pestaña mediana',
        id: '8',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'default',
        content: 'contenito 1'
    },
    {
        name: 'Pestaña mediana',
        id: '9',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'default',
        content: 'Contenido Pestaña 2'
    },
    {
        name: 'Pestaña mediana',
        id: '10',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'default',
        content: 'Contenido Pestaña 3'
    }
];
var sizeSmItems = [
    {
        name: 'Pestaña chica',
        id: '11',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'small',
        content: 'Contenido Pestaña 1'
    },
    {
        name: 'Pestaña chica',
        id: '12',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'small',
        content: 'Contenido Pestaña 2'
    },
    {
        name: 'Pestaña chica',
        id: '13',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'small',
        content: 'Contenido Pestaña 3'
    }
];
var sliderItems = [
    {
        name: 'Pestaña 1',
        id: '14',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  1'
    },
    {
        name: 'Pestaña 2',
        id: '15',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  2'
    },
    {
        name: 'Pestaña 3',
        id: '16',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  3'
    },
    {
        name: 'Pestaña 4',
        id: '17',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  4'
    },
    {
        name: 'Pestaña 5',
        id: '18',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  5'
    },
    {
        name: 'Pestaña 6',
        id: '19',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  6'
    },
    {
        name: 'Pestaña 7',
        id: '20',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  7'
    },
    {
        name: 'Pestaña 8',
        id: '21',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  8'
    },
    {
        name: 'Pestaña 9',
        id: '22',
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  9'
    },
    {
        name: 'Pestaña 10',
        id: '23',
        disabled: true,
        iconTabs: '<span class="material-icons-round">home</span>',
        content: 'Contenido Pestaña  10'
    }
];
exports.ContainerTabs = function () {
    return (react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box", isWithButton: false, idPrefix: "simple", label: "Simple tab list" }));
};
exports.ContainerTabs.story = {
    name: 'Ancho variable'
};
exports.ContainerTabWidth = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box fixedWidth", isWithButton: false, idPrefix: "width", label: "width tab list" })));
};
exports.ContainerTabWidth.story = {
    name: 'Ancho fijo'
};
exports.ContainerSizeTabs = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeLgItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box", isWithButton: false, idPrefix: "sizeLg", label: "lg tab list" }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box", isWithButton: false, idPrefix: "sizeMd", label: "md tab list" }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeSmItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box", isWithButton: false, idPrefix: "sizeSm", label: "sm tab list" })));
};
exports.ContainerSizeTabs.story = {
    name: 'Tamaños'
};
exports.ContainerTabsBg = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-bg" },
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs nav-box-bg", isWithButton: false, idPrefix: "container", label: "container tab list" })));
};
exports.ContainerTabsBg.story = {
    name: 'Con fondo'
};
exports.ContainerTabsIcon = function () {
    return (react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: true, classUl: "nav nav-pills tabs nav-box", isWithButton: false, idPrefix: "icon", label: "icon tab list" }));
};
exports.ContainerTabsIcon.story = {
    name: 'Con ícono'
};
exports.ContainerSliderTabs = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-tabs" },
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sliderItems, selected: "1", hasIcon: true, classUl: "nav nav-pills tabs nav-box", idPrefix: "containerSlider", label: "container slider tab list " })));
};
exports.ContainerSliderTabs.story = {
    name: 'Desplazable'
};
exports.ContainerSliderTabBg = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-bg" },
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sliderItems, selected: "1", hasIcon: true, classUl: "nav nav-pills tabs nav-box-bg", idPrefix: "containerSliderBg", label: "container background slider tab list " })));
};
exports.ContainerSliderTabBg.story = {
    name: 'Desplazable con fondo'
};
