// @gcba/obelisco/dist/components/Nav/TabsSimple.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SliderTab = exports.SimpleIcon = exports.SimpleSizes = exports.SimpleWidth = exports.Simple = void 0;
var react_1 = __importDefault(require("react"));
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var tabs_1 = require("./tabs");
// Configuración general del componente
exports.default = {
    component: _1.Nav,
    title: 'Componentes|Navegación y pestañas/Pestañas simple',
    decorators: [addon_a11y_1.withA11y]
};
var simpleItems = [
    { name: 'Pestaña 1', id: '1', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 2', id: '2', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 3', id: '3', iconTabs: '<span class="material-icons-round">home</span>' },
    {
        name: 'Pestaña 4',
        id: '4',
        disabled: true,
        iconTabs: '<span class="material-icons-round">home</span>'
    }
];
var sizeLgItems = [
    {
        name: 'Pestaña grande',
        id: '1',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'large'
    },
    { name: 'Pestaña grande', id: '2', iconTabs: '<span class="material-icons-round">home</span>', type: 'large' },
    {
        name: 'Pestaña grande',
        id: '3',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'large'
    }
];
var sizeItems = [
    {
        name: 'Pestaña mediana',
        id: '1',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'default'
    },
    { name: 'Pestaña mediana', id: '2', iconTabs: '<span class="material-icons-round">home</span>', type: 'default' },
    {
        name: 'Pestaña mediana',
        id: '3',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'default'
    }
];
var sizeSmItems = [
    {
        name: 'Pestaña chica',
        id: '1',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'small'
    },
    { name: 'Pestaña chica', id: '2', iconTabs: '<span class="material-icons-round">home</span>', type: 'small' },
    {
        name: 'Pestaña chica',
        id: '3',
        iconTabs: '<span class="material-icons-round">home</span>',
        type: 'small'
    }
];
var sliderItems = [
    { name: 'Pestaña 1', id: '1', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 2', id: '2', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 3', id: '3', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 4', id: '4', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 5', id: '5', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 6', id: '6', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 7', id: '7', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 8', id: '8', iconTabs: '<span class="material-icons-round">home</span>' },
    { name: 'Pestaña 9', id: '9', iconTabs: '<span class="material-icons-round">home</span>' },
    {
        name: 'Pestaña 10',
        id: '10',
        disabled: true,
        iconTabs: '<span class="material-icons-round">home</span>'
    }
];
exports.Simple = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs", isWithButton: false })));
};
exports.Simple.story = {
    name: 'Ancho variable'
};
exports.SimpleWidth = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs fixedWidth", isWithButton: false })));
};
exports.SimpleWidth.story = {
    name: 'Ancho fijo'
};
exports.SimpleSizes = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeLgItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs", isWithButton: false }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs", isWithButton: false }),
        react_1.default.createElement("br", null),
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sizeSmItems, selected: "1", hasIcon: false, classUl: "nav nav-pills tabs", isWithButton: false })));
};
exports.SimpleSizes.story = {
    name: 'Tamaños'
};
exports.SimpleIcon = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: simpleItems, selected: "1", hasIcon: true, classUl: "nav nav-pills tabs", isWithButton: false })));
};
exports.SimpleIcon.story = {
    name: 'Con ícono'
};
exports.SliderTab = function () {
    return (react_1.default.createElement("div", { className: "wrapper-tabs" },
        react_1.default.createElement(tabs_1.NavTabsSlider, { items: sliderItems, selected: "1", hasIcon: true, classUl: "nav nav-pills tabs" })));
};
exports.SliderTab.story = {
    name: 'Desplazable'
};
