// @gcba/obelisco/dist/components/Button/Button.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ButtonWithSpinner = exports.ButtonWithIcon = exports.ButtonBlock = exports.ButtonSizes = exports.ButtonTypes = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Button.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var utils_1 = require("../utils");
var _1 = require(".");
var Spinner_1 = require("../Spinner");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Botones',
    component: _1.Button,
    decorators: [addon_a11y_1.withA11y]
};
exports.ButtonTypes = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "storybook__container-btn" },
            utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type }, item.name)); }),
            react_1.default.createElement("a", { className: "btn btn-primary", href: "#", target: "_blank" }, "Enlace")),
        react_1.default.createElement("div", { className: "storybook__container-btn" },
            utils_1.buttonTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, isOutline: true }, item.name)); }),
            react_1.default.createElement("a", { className: "btn btn-outline-primary", href: "#", target: "_blank" }, "Enlace"))));
};
exports.ButtonTypes.story = { name: 'Tipos' };
exports.ButtonSizes = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-btn" },
        utils_1.sizesButton.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.size, type: "secondary", size: item.size }, item.name)); }),
        react_1.default.createElement("a", { className: "btn btn-secondary", href: "#", target: "_blank" }, "Enlace")));
};
exports.ButtonSizes.story = { name: 'Tamaños' };
exports.ButtonBlock = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-btn-block" },
        react_1.default.createElement(_1.Button, { type: "secondary", isBlock: true }, "Expandible")));
};
exports.ButtonBlock.story = { name: 'Expandible' };
exports.ButtonWithIcon = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "storybook__container-btn" },
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", size: size, isIconSpinner: react_1.default.createElement("i", { className: "bx bxs-info-circle" }) }, "Bot\u00F3n")); }),
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", isOutline: true, size: size, isIconSpinner: react_1.default.createElement("span", { className: "material-icons-round" }, "info") }, "Bot\u00F3n")); })),
        react_1.default.createElement("div", { className: "storybook__container-btn" },
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", size: size, isIconSpinner: react_1.default.createElement("i", { className: "bx bxs-info-circle" }), ariaLabel: "Bot\u00F3n" })); }),
            utils_1.sizes.map(function (size) { return (react_1.default.createElement(_1.Button, { key: size, type: "primary", isOutline: true, size: size, isIconSpinner: react_1.default.createElement("span", { className: "material-icons-round" }, "info"), ariaLabel: "Bot\u00F3n" })); }))));
};
exports.ButtonWithIcon.story = { name: 'Con ícono' };
exports.ButtonWithSpinner = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "storybook__container-btn" }, utils_1.buttonSpinnerTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, isIconSpinner: react_1.default.createElement(Spinner_1.Spinner, { type: item.spinner, size: "small" }) }, "Bot\u00F3n")); })),
    react_1.default.createElement("div", { className: "storybook__container-btn" }, utils_1.buttonOutlineSpinnerTypes.map(function (item) { return (react_1.default.createElement(_1.Button, { key: item.type, type: item.type, isOutline: true, isIconSpinner: react_1.default.createElement(Spinner_1.Spinner, { type: item.spinner, size: "small" }) }, "Bot\u00F3n")); })))); };
exports.ButtonWithSpinner.story = { name: 'Con spinner' };
