// @gcba/obelisco/dist/components/List-group/AccesListColumn.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnaColumna = exports.DosColumnas = exports.TresColumnas = exports.CuatroColumnas = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./List-group.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var index_1 = require("./index");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Accesos/Lista ancho por acceso',
    decorators: [addon_a11y_1.withA11y]
};
var type = 'item-sm';
var name = '';
exports.CuatroColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "access-deck access-column max-cards-4" },
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }))));
};
exports.TresColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "access-deck access-column max-cards-3" },
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }))));
};
exports.DosColumnas = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "access-deck access-column max-cards-2" },
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }),
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }))));
};
exports.UnaColumna = function () {
    return (react_1.default.createElement("div", { className: "container" },
        react_1.default.createElement("div", { className: "access-deck access-column max-cards-1" },
            react_1.default.createElement(index_1.AccessSizesComponent, { type: type, name: name }))));
};
