// @gcba/obelisco/dist/components/Dropdown/Dropdown.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.DropdownNested = exports.DropdownPosition = exports.DropdownIcons = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Dropdown.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var index_1 = require("./index");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Desplegables/De navegación',
    decorators: [addon_a11y_1.withA11y]
};
exports.DropdownIcons = function () { return (react_1.default.createElement("div", { className: "storybook__container-dropdown" },
    react_1.default.createElement(index_1.Dropdown, { btnIconRight: "expand_more", btnIconRightClass: "btn-dropdown-icon", isBordered: true },
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "info", itemIconRight: "info" }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "info", itemIconRight: "info", isAlignRight: true }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "info", itemIconRight: "info", isAlignRight: true, isDisabled: true }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "logout", isDanger: true })),
    react_1.default.createElement(index_1.Dropdown, { btnIconLeft: "menu", isBordered: true },
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "info" }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "info", isAlignRight: true }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "logout", isDanger: true, isAlignRight: true })),
    react_1.default.createElement(index_1.Dropdown, { btnIconLeft: "menu", onlyIcon: true, isBordered: true },
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "info" }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "info", isAlignRight: true }),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "logout", isDanger: true, isAlignRight: true })))); };
exports.DropdownIcons.story = {
    name: 'Con íconos'
};
exports.DropdownPosition = function () { return (react_1.default.createElement("div", { className: "storybook__container-dropdown-position" },
    react_1.default.createElement(index_1.Dropdown, { title: "Desplegable Izq.", btnIconRight: "expand_more", btnIconRightClass: "btn-dropdown-icon", isBordered: true },
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "logout", isDanger: true })),
    react_1.default.createElement(index_1.Dropdown, { title: "Desplegable Der.", btnIconLeft: "expand_more", btnIconLeftClass: "btn-dropdown-icon", isBordered: true, isMenuRight: true },
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "logout", isDanger: true, isAlignRight: true })))); };
exports.DropdownPosition.story = {
    name: 'Disposición'
};
exports.DropdownNested = function () { return (react_1.default.createElement("div", { className: "storybook__container-dropdown-position" },
    react_1.default.createElement(index_1.Dropdown, { title: "Desplegable Izq.", btnIconRight: "expand_more", btnIconRightClass: "btn-dropdown-icon", isBordered: true, isNested: true, idDropdown: "dropdownContentLeft" },
        react_1.default.createElement(index_1.Dropdown, { title: "Opci\u00F3n en cascada", isSubDropdown: true, btnIconRight: "expand_more", btnIconRightClass: "btn-dropdown-icon" },
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true }),
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true }),
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true })),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, { itemIconLeft: "logout", isDanger: true })),
    react_1.default.createElement(index_1.Dropdown, { title: "Desplegable Der.", btnIconLeft: "expand_more", btnIconLeftClass: "btn-dropdown-icon", isBordered: true, isNested: true, idDropdown: "dropdownContentRight", isMenuRight: true },
        react_1.default.createElement(index_1.Dropdown, { title: "Opci\u00F3n en cascada", isSubDropdown: true, btnIconLeft: "expand_more", btnIconLeftClass: "btn-dropdown-icon", isMenuRight: true },
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true }),
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true }),
            react_1.default.createElement(index_1.DropdownItem, { isSubItem: true })),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, null),
        react_1.default.createElement(index_1.DropdownItem, { itemIconRight: "logout", isDanger: true })))); };
exports.DropdownNested.story = {
    name: 'Con cascada'
};
