// @gcba/obelisco/dist/components/Dropdown/Dropdown.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Idioma = exports.SoloIcono = exports.DesplegablePequeno = exports.DesplegableGrande = void 0;
// Base
var react_1 = require("react");
require("./Dropdown.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración del general del componente
exports.default = {
    title: 'Componentes | Desplegables',
    decorators: [addon_a11y_1.withA11y]
};
exports.DesplegableGrande = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-lg", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            "Mi perfil ",
            react_1.default.createElement("span", { className: "dropdown-user-icon" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))))); };
exports.DesplegablePequeno = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "dropdown-container" },
        react_1.default.createElement("div", { className: "dropdown" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-sm dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Filtro"),
            react_1.default.createElement("div", { className: "dropdown-menu" },
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
        react_1.default.createElement("div", { className: "dropdown" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-sm dropdown-toggle active", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Filtro (1)"),
            react_1.default.createElement("div", { className: "dropdown-menu" },
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3")))))); };
exports.DesplegablePequeno.story = { name: 'Desplegable pequeño' };
exports.SoloIcono = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            react_1.default.createElement("span", { className: "dropdown-user-icon ml-0" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))))); };
exports.Idioma = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            "ES ",
            react_1.default.createElement("span", { className: "dropdown-language-icon" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Ingl\u00E9s"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Espa\u00F1ol"))))); };
