// @gcba/obelisco/dist/components/Dropdown/Dropdown.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Tamanos = exports.Idioma = exports.SoloIcono = exports.DesplegableConIcono = exports.DesplegableConEnlaces = exports.Desplegable = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Dropdown.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Desplegables',
    decorators: [addon_a11y_1.withA11y]
};
exports.Desplegable = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "dropdown-container" },
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-sm dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable"),
            react_1.default.createElement("div", { className: "dropdown-menu" },
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-sm dropdown-toggle active", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable (1)"),
            react_1.default.createElement("div", { className: "dropdown-menu" },
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-sm dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable (1)"),
            react_1.default.createElement("div", { className: "dropdown-menu" },
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
                react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3")))))); };
exports.Desplegable.story = {
    name: 'Desplegable botón'
};
exports.DesplegableConEnlaces = function () { return (react_1.default.createElement(react_1.default.Fragment, null,
    react_1.default.createElement("div", { className: "dropdown-container" },
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-sm dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable"),
            react_1.default.createElement("ul", { className: "dropdown-menu" },
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 1")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 2")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 3")))),
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable"),
            react_1.default.createElement("ul", { className: "dropdown-menu" },
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 1")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 2")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 3")))),
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-lg dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable"),
            react_1.default.createElement("ul", { className: "dropdown-menu" },
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 1")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 2")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 3")))),
        react_1.default.createElement("div", { className: "dropdown mb-3" },
            react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-lg dropdown-toggle active", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Desplegable"),
            react_1.default.createElement("ul", { className: "dropdown-menu" },
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 1")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 2")),
                react_1.default.createElement("li", null,
                    react_1.default.createElement("a", { className: "dropdown-item", href: "#" }, "Opci\u00F3n 3"))))))); };
exports.DesplegableConEnlaces.story = {
    name: 'Desplegable enlace'
};
exports.DesplegableConIcono = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown mb-3" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-lg", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            "Mi perfil",
            react_1.default.createElement("span", { className: "dropdown-user-icon" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            "Mi perfil",
            react_1.default.createElement("span", { className: "dropdown-user-icon" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))))); };
exports.DesplegableConIcono.story = {
    name: 'Desplegable con ícono'
};
exports.SoloIcono = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown", "aria-label": "Dropdown", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            react_1.default.createElement("span", { className: "dropdown-user-icon ml-0" }, "\u00A0")),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))))); };
exports.SoloIcono.story = {
    name: 'Desplegable solo ícono'
};
exports.Idioma = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" },
            "ES ",
            react_1.default.createElement("span", { className: "dropdown-language-icon" })),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Ingl\u00E9s"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Espa\u00F1ol"))))); };
exports.Idioma.story = {
    name: 'Desplegable idioma'
};
exports.Tamanos = function () { return (react_1.default.createElement("div", { className: "dropdown-container" },
    react_1.default.createElement("div", { className: "dropdown mb-3" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-lg dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Filtro (1)"),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
    react_1.default.createElement("div", { className: "dropdown mb-3" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Filtro (1)"),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))),
    react_1.default.createElement("div", { className: "dropdown mb-3" },
        react_1.default.createElement("button", { type: "button", className: "btn btn-dropdown btn-dropdown-border btn-dropdown-sm dropdown-toggle", "data-toggle": "dropdown", "aria-haspopup": "true", "aria-expanded": "false" }, "Filtro (1)"),
        react_1.default.createElement("div", { className: "dropdown-menu" },
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 1"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 2"),
            react_1.default.createElement("button", { className: "dropdown-item", type: "button" }, "Opci\u00F3n 3"))))); };
exports.Tamanos.story = { name: 'Tamaños' };
