// @gcba/obelisco/dist/components/utils.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.buttonOutlineSpinnerTypes = exports.buttonSpinnerTypes = exports.spinnerTypes = exports.directionBottom = exports.directionRight = exports.directionLeft = exports.directionTop = exports.sizeToClass = exports.sizesButton = exports.sizes = exports.buttonTypes = exports.types = void 0;
exports.types = ['primary', 'success', 'danger'];
exports.buttonTypes = [
    { type: 'primary', name: 'Primario' },
    { type: 'secondary', name: 'Secundario' },
    { type: 'success', name: 'Éxito' },
    { type: 'danger', name: 'Error' },
    { type: 'link', name: 'Link' }
];
exports.sizes = ['small', 'default', 'large'];
exports.sizesButton = [
    {
        size: 'small',
        name: 'Chico'
    },
    {
        size: 'default',
        name: 'Mediano'
    },
    {
        size: 'large',
        name: 'Grande'
    }
];
exports.sizeToClass = function (size) {
    switch (size) {
        case 'default':
            return '';
        case 'large':
            return 'lg';
        case 'small':
            return 'sm';
    }
};
exports.directionTop = [
    { direction: 'top-right', icon: 'turn_right', position: 'arriba-derecha' },
    { direction: 'top', icon: 'arrow_upward', position: 'arriba' },
    { direction: 'top-left', icon: 'turn_left', position: 'arriba-izquierda' }
];
exports.directionLeft = [
    { direction: 'right-bottom', icon: 'south_east', position: 'derecha-abajo' },
    { direction: 'right', icon: 'arrow_forward', position: 'derecha' },
    { direction: 'right-top', icon: 'north_east', position: 'derecha-arriba' }
];
exports.directionRight = [
    { direction: 'left-bottom', icon: 'south_west', position: 'izquierda-abajo' },
    { direction: 'left', icon: 'keyboard_backspace', position: 'izquierda' },
    { direction: 'left-top', icon: 'north_west', position: 'izquierda-arriba' }
];
exports.directionBottom = [
    { direction: 'bottom-right', icon: 'subdirectory_arrow_right', position: 'abajo-derecha' },
    { direction: 'bottom', icon: 'arrow_downward', position: 'abajo' },
    { direction: 'bottom-left', icon: 'subdirectory_arrow_left', position: 'abajo-izquierda' }
];
exports.spinnerTypes = ['primary', 'secondary', 'success', 'danger', 'info', 'light', 'dark'];
exports.buttonSpinnerTypes = [
    { type: 'primary', spinner: 'dark' },
    { type: 'secondary', spinner: 'light' },
    { type: 'success', spinner: 'light' },
    { type: 'danger', spinner: 'light' },
    { type: 'link', spinner: 'info' }
];
exports.buttonOutlineSpinnerTypes = [
    { type: 'primary', spinner: 'dark' },
    { type: 'secondary', spinner: 'secondary' },
    { type: 'success', spinner: 'success' },
    { type: 'danger', spinner: 'danger' },
    { type: 'link', spinner: 'info' }
];
