// @gcba/obelisco/dist/components/Banner/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
    Object.defineProperty(o, "default", { enumerable: true, value: v });
}) : function(o, v) {
    o["default"] = v;
});
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (k !== "default" && Object.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
    __setModuleDefault(result, mod);
    return result;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.BUTTONS_BANNER = exports.BUTTON_BANNER = exports.TEXT_BANNER_LINK = exports.TextBannerButton = exports.handleTextResponsive = exports.Banner = void 0;
var react_1 = __importStar(require("react"));
var classnames_1 = __importDefault(require("classnames"));
exports.Banner = function (props) {
    var _a;
    var isDark = props.isDark, buttons = props.buttons, iconBanner = props.iconBanner, textBanner = props.textBanner;
    var BannerClasses = classnames_1.default('banner', (_a = {},
        _a['banner-dark'] = isDark !== undefined,
        _a));
    return (react_1.default.createElement("div", { className: BannerClasses.trim() },
        react_1.default.createElement("div", { className: "banner-content" },
            iconBanner && !(iconBanner === null || iconBanner === void 0 ? void 0 : iconBanner.includes('bx')) && react_1.default.createElement("span", { className: "material-icons-round" }, iconBanner),
            iconBanner && (iconBanner === null || iconBanner === void 0 ? void 0 : iconBanner.includes('bx')) && react_1.default.createElement("i", { className: iconBanner }),
            react_1.default.createElement("p", { className: "banner-text" }, textBanner),
            buttons && (react_1.default.createElement("div", { className: "banner-actions" }, buttons === null || buttons === void 0 ? void 0 : buttons.map(function (_a, index) {
                var className = _a.className, name = _a.name, url = _a.url;
                return (react_1.default.createElement("a", { key: index, href: url, target: "_blank", rel: "noreferrer", className: className }, name));
            }))))));
};
exports.handleTextResponsive = function () {
    var _a = react_1.useState(false), textResponsive = _a[0], setTextResponsive = _a[1];
    react_1.useEffect(function () {
        function handleResize() {
            setTextResponsive(window.innerWidth < 752);
        }
        window.addEventListener('resize', handleResize);
        handleResize();
        return function () { return window.removeEventListener('resize', handleResize); };
    }, []);
    return textResponsive;
};
exports.TextBannerButton = function (props) {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        "Este banner tiene ",
        props.textButtonAmount,
        " y ",
        react_1.default.createElement("strong", null, "puede tener \u00E9nfasis"),
        ". Se truncan los textos que superen ",
        exports.handleTextResponsive() ? '3 líneas de texto' : 'más de una línea de texto',
        "."));
};
exports.TEXT_BANNER_LINK = (react_1.default.createElement(react_1.default.Fragment, null,
    "Esta descripci\u00F3n de banner ",
    react_1.default.createElement("strong", null, "puede tener \u00E9nfasis"),
    " y",
    ' ',
    react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "enlace"),
    "."));
exports.BUTTON_BANNER = [
    {
        name: 'Botón',
        className: 'btn btn-sm btn-secondary',
        url: '#'
    }
];
exports.BUTTONS_BANNER = [
    {
        name: 'Botón',
        className: 'btn btn-sm btn-secondary',
        url: '#'
    },
    {
        name: 'Botón',
        className: 'btn btn-sm btn-primary',
        url: '#'
    }
];
