// @gcba/obelisco/dist/components/Panel/HighlightedPanelBanner.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.WithBackground = exports.WithoutMultimedia = exports.WithVideo = exports.WithImage = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var Panel_constants_1 = require("./Panel.constants");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Destacado/Banner',
    decorators: [addon_a11y_1.withA11y]
};
exports.WithImage = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n"),
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg btn-icon", href: "#", target: "_blank", download: true },
                react_1.default.createElement("span", { className: "material-icons-round" }, "file_download"),
                "Descargar")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank" }, "Enlace externo")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED },
            react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                react_1.default.createElement("div", { className: "access-content" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso")))),
        react_1.default.createElement("br", null)));
};
exports.WithImage.story = { name: 'Con imagen' };
exports.WithVideo = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, video: Panel_constants_1.VIDEO },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n"),
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, iframe: Panel_constants_1.IFRAME },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n"),
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg", href: "#" }, "Bot\u00F3n"))));
};
exports.WithVideo.story = { name: 'Con video' };
exports.WithoutMultimedia = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n"),
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg btn-icon", href: "#", target: "_blank", download: true },
                react_1.default.createElement("span", { className: "material-icons-round" }, "file_download"),
                "Descargar")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED },
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank" }, "Enlace externo")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED },
            react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                react_1.default.createElement("div", { className: "access-content" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso")))),
        react_1.default.createElement("br", null)));
};
exports.WithoutMultimedia.story = { name: 'Sin Multimedia' };
exports.WithBackground = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED, bgColor: "light" },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n"),
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED, bgColor: "light" },
            react_1.default.createElement("a", { className: "btn btn-primary btn-lg", href: "#" }, "Bot\u00F3n")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED, bgColor: "light" },
            react_1.default.createElement("a", { className: "btn btn-secondary btn-lg btn-icon", href: "#", target: "_blank", download: true },
                react_1.default.createElement("span", { className: "material-icons-round" }, "file_download"),
                "Descargar")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED, bgColor: "light" },
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank" }, "Enlace externo")),
        react_1.default.createElement("br", null),
        react_1.default.createElement(_1.HighlightedPanel, { title: Panel_constants_1.TITLE_HIGHLIGHTED, description: Panel_constants_1.DESCRIPTION_HIGHLIGHTED, picture: Panel_constants_1.PICTURE_HIGHLIGHTED, bgColor: "light" },
            react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                react_1.default.createElement("div", { className: "access-content" },
                    react_1.default.createElement("span", { className: "access-title" }, "Acceso")))),
        react_1.default.createElement("br", null)));
};
exports.WithBackground.story = { name: 'Con fondo' };
