// @gcba/obelisco/dist/components/Panel/PanelImageHorizontal.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.HighLightPanelLink = exports.HighLightPanelButtonLink = exports.HighLightPanel = void 0;
// Addons
var react_1 = require("react");
var addon_a11y_1 = require("@storybook/addon-a11y");
var _1 = require(".");
exports.default = {
    title: 'Componentes|Panel/Panel destacado grande horizontal/Con Imagen',
    decorators: [addon_a11y_1.withA11y]
};
var TITLE = 'Encabezado';
var DESCRIPTION = 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Suscipit nostrum quo animi dignissimos omnis dicta quia placeat, nesciunt quasi hic autem iure eum necessitatibus ea expedita quas. Rem, aspernatur repudiandae.';
var BUTTONS_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    },
    {
        name: 'Botón',
        className: 'btn btn-lg btn-secondary card-link'
    }
];
var LINK_BUTTON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg download-link'
};
var LINK = {
    name: 'Descargar',
    className: 'download'
};
exports.HighLightPanel = function () {
    return (react_1.default.createElement(_1.PanelHorizontal, { title: TITLE, description: DESCRIPTION, picture: {
            src: 'panel/larreta.jpg',
            alt: 'descripción alternativa'
        }, buttons: BUTTONS_ARR }));
};
exports.HighLightPanel.story = { name: 'Con botones' };
exports.HighLightPanelButtonLink = function () {
    return (react_1.default.createElement(_1.PanelHorizontal, { title: TITLE, description: DESCRIPTION, picture: {
            src: 'panel/larreta.jpg',
            alt: 'descripción alternativa'
        }, link: LINK_BUTTON }));
};
exports.HighLightPanelButtonLink.story = { name: 'Con botón de descarga' };
exports.HighLightPanelLink = function () {
    return (react_1.default.createElement(_1.PanelHorizontal, { title: TITLE, description: DESCRIPTION, picture: {
            src: 'panel/larreta.jpg',
            alt: 'descripción alternativa'
        }, link: LINK }));
};
exports.HighLightPanelLink.story = { name: 'Con enlace de descarga' };
