// @gcba/obelisco/dist/components/Panel/Panel.constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.LINK_LIST_BUTTONS = exports.LINK_DOWNLOAD_LIST = exports.LINK_LIST = exports.SMALL_LINK_DOWNLOAD = exports.SMALL_PICTURE_XS = exports.SMALL_PICTURE = exports.SMALL_BUTTONS_ARR = exports.IFRAME = exports.VIDEO = exports.PICTURE_HIGHLIGHTED = exports.PICTURE = exports.LINK_DOWNLOAD_BUTTON_ICON = exports.LINK_DOWNLOAD_BUTTON = exports.LINK_DOWNLOAD = exports.LINK = exports.BUTTONS_ARR = exports.BUTTON_ARR = exports.DESCRIPTION_HIGHLIGHTED_LATERAL = exports.DESCRIPTION_HIGHLIGHTED = exports.TITLE_HIGHLIGHTED = exports.DESCRIPTION = exports.TITLE = void 0;
/* Large panel */
exports.TITLE = 'Nombre del titular';
exports.DESCRIPTION = 'Descripción de hasta 3 líneas para mantener los estándares de lectura en los componentes de Obelisco.';
exports.TITLE_HIGHLIGHTED = 'Título del destacado';
exports.DESCRIPTION_HIGHLIGHTED = 'Este es el cuerpo de un destacado. Debe ser breve y conciso, de pocas líneas. No puede contener negritas ni enlaces. Este es el cuerpo de un destacado. Debe ser breve y conciso, de pocas líneas. No puede contener negritas ni enlaces';
exports.DESCRIPTION_HIGHLIGHTED_LATERAL = 'Este es el cuerpo de un destacado. Debe ser breve y conciso, de pocas líneas. No puede contener negritas ni enlaces.';
exports.BUTTON_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    }
];
exports.BUTTONS_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    },
    {
        name: 'Botón',
        className: 'btn btn-lg btn-secondary card-link'
    }
];
exports.LINK = {
    name: 'Enlace',
    url: '#'
};
exports.LINK_DOWNLOAD = {
    name: 'Descargar',
    className: 'download',
    isDownload: true
};
exports.LINK_DOWNLOAD_BUTTON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg download-link',
    isDownload: true
};
exports.LINK_DOWNLOAD_BUTTON_ICON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg btn-icon card-link',
    icon: 'file_download',
    isDownload: true
};
exports.PICTURE = {
    src: 'panel/hlarreta.jpg',
    alt: 'descripción alternativa'
};
exports.PICTURE_HIGHLIGHTED = {
    src: 'panel/destacado.jpeg',
    alt: 'descripción de imagen'
};
exports.VIDEO = {
    src: 'panel/videoBuenosAires.mp4',
    title: 'Buenos Aires se escribe en plural',
    caption: 'panel/videoBuenosAires.vtt'
};
exports.IFRAME = {
    src: 'https://www.youtube.com/embed/sXE613Oaxvc?si=iWSX1erqQxXOLojw',
    title: 'Buenos Aires se escribe en plural'
};
/* Small Panel */
exports.SMALL_BUTTONS_ARR = [
    {
        name: 'Acción 1',
        className: 'btn btn-secondary btn-sm card-link'
    },
    {
        name: 'Acción 2',
        className: 'btn btn-link btn-sm card-link'
    }
];
exports.SMALL_PICTURE = {
    src: 'cards/paseobajo-wide.png',
    alt: 'descripción alternativa',
    className: 'rounded-lg',
    size: 154
};
exports.SMALL_PICTURE_XS = {
    src: 'cards/paseobajo-wide.png',
    alt: 'descripción alternativa',
    className: 'rounded-lg',
    size: 104
};
exports.SMALL_LINK_DOWNLOAD = {
    name: 'Descargar',
    className: 'download',
    url: '#',
    isDownload: true
};
exports.LINK_LIST = {
    listTitle: 'Título de una lista de enlaces',
    links: [
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' },
        { name: 'Enlace predeterminado', url: '#' }
    ]
};
exports.LINK_DOWNLOAD_LIST = {
    listTitle: 'Título de una lista de enlaces',
    links: [
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true },
        { name: 'Enlace de descarga', className: 'download', url: '#', isDownload: true }
    ]
};
exports.LINK_LIST_BUTTONS = [
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    },
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    },
    {
        listTitle: 'Nombre del archivo',
        name: 'Descargar',
        className: 'btn btn-secondary btn-sm download-link card-link',
        url: '#',
        isDownload: true
    }
];
