// @gcba/obelisco/dist/components/Panel/PanelSmallVertical.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.SmallPanelVerticalBackground = exports.SmallPanelVerticalListDownloadButton = exports.SmallPanelVerticalListLinkDownload = exports.SmallPanelVerticalListLink = exports.SmallPanelVerticalLinkDownload = exports.SmallPanelVerticalLink = exports.SmallPanelVerticalButtons = exports.SmallPanelVerticalWithoutAccion = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Panel.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var Panel_constants_1 = require("./Panel.constants");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Panel/Chico/Vertical',
    decorators: [addon_a11y_1.withA11y]
};
exports.SmallPanelVerticalWithoutAccion = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE })));
};
exports.SmallPanelVerticalWithoutAccion.story = { name: 'Sin accionable' };
exports.SmallPanelVerticalButtons = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, buttons: Panel_constants_1.SMALL_BUTTONS_ARR })));
};
exports.SmallPanelVerticalButtons.story = { name: 'Con botones' };
exports.SmallPanelVerticalLink = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, link: Panel_constants_1.LINK })));
};
exports.SmallPanelVerticalLink.story = { name: 'Con enlace' };
exports.SmallPanelVerticalLinkDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, link: Panel_constants_1.SMALL_LINK_DOWNLOAD })));
};
exports.SmallPanelVerticalLinkDownload.story = { name: 'Con enlace de descarga' };
exports.SmallPanelVerticalListLink = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, listLinkPanel: Panel_constants_1.LINK_LIST })));
};
exports.SmallPanelVerticalListLink.story = { name: 'Con lista de enlaces' };
exports.SmallPanelVerticalListLinkDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, listLinkPanel: Panel_constants_1.LINK_DOWNLOAD_LIST })));
};
exports.SmallPanelVerticalListLinkDownload.story = { name: 'Con lista de enlaces de descarga' };
exports.SmallPanelVerticalListDownloadButton = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, listLinkButtonPanel: Panel_constants_1.LINK_LIST_BUTTONS })));
};
exports.SmallPanelVerticalListDownloadButton.story = { name: 'Con lista de botón de descarga' };
exports.SmallPanelVerticalBackground = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-small" },
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, buttons: Panel_constants_1.SMALL_BUTTONS_ARR, bgColor: "light" }),
        react_1.default.createElement(_1.SmallPanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.SMALL_PICTURE, buttons: Panel_constants_1.SMALL_BUTTONS_ARR, bgColor: "white" })));
};
exports.SmallPanelVerticalBackground.story = { name: 'Con fondo' };
