// @gcba/obelisco/dist/components/Panel/PanelLargeHorizontal.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.HorizontalPanelBackground = exports.HorizontalPanelLinkButtonDownload = exports.HorizontalPanelLinkDownload = exports.HorizontalPanelLink = exports.HorizontalPanelButton = exports.HorizontalPanelButtons = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
var Panel_constants_1 = require("./Panel.constants");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Panel/Grande/Horizontal',
    decorators: [addon_a11y_1.withA11y]
};
/* Con Imagen | Video y 2 botones */
exports.HorizontalPanelButtons = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, buttons: Panel_constants_1.BUTTONS_ARR })));
};
exports.HorizontalPanelButtons.story = { name: 'Con dos botones' };
/* Con Imagen | Video y 1 boton */
exports.HorizontalPanelButton = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTON_ARR }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, buttons: Panel_constants_1.BUTTON_ARR })));
};
exports.HorizontalPanelButton.story = { name: 'Con un botón' };
/* Con Imagen | Video y enlace */
exports.HorizontalPanelLink = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK })));
};
exports.HorizontalPanelLink.story = { name: 'Con enlace' };
/* Con Imagen | Video y enlace de descarga */
exports.HorizontalPanelLinkDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK_DOWNLOAD }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK_DOWNLOAD })));
};
exports.HorizontalPanelLinkDownload.story = { name: 'Con enlace de descarga' };
/* Con Imagen | Video y boton de descarga */
exports.HorizontalPanelLinkButtonDownload = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, link: Panel_constants_1.LINK_DOWNLOAD_BUTTON }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, video: Panel_constants_1.VIDEO, link: Panel_constants_1.LINK_DOWNLOAD_BUTTON_ICON })));
};
exports.HorizontalPanelLinkButtonDownload.story = { name: 'Con botón de descarga' };
/* Con fondo bris y blanco*/
exports.HorizontalPanelBackground = function () {
    return (react_1.default.createElement("div", { className: "storybook__wrapper-horizontal" },
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR, bgColor: "light" }),
        react_1.default.createElement(_1.LargePanel, { title: Panel_constants_1.TITLE, description: Panel_constants_1.DESCRIPTION, picture: Panel_constants_1.PICTURE, buttons: Panel_constants_1.BUTTONS_ARR, bgColor: "white" })));
};
exports.HorizontalPanelBackground.story = { name: 'Con fondo' };
