// @gcba/obelisco/dist/components/Panel/PanelVideoVertical.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.HighLightPanelLink = exports.HighLightPanelButtonLink = exports.HighLightPanel = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Panel/Panel vertical/Con video',
    decorators: [addon_a11y_1.withA11y]
};
var TITLE = 'Encabezado';
var DESCRIPTION = 'Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.';
var BUTTONS_ARR = [
    {
        name: 'Botón',
        className: 'btn btn-lg btn-primary card-link'
    },
    {
        name: 'Botón',
        className: 'btn btn-lg btn-secondary card-link'
    }
];
var LINK_BUTTON = {
    name: 'Descargar',
    className: 'btn btn-secondary btn-lg download-link'
};
var LINK = {
    name: 'Descargar',
    className: 'download'
};
exports.HighLightPanel = function () {
    return (react_1.default.createElement("div", { className: "panel-wrapper" },
        react_1.default.createElement(_1.PanelVertical, { title: TITLE, description: DESCRIPTION, video: {
                src: 'panel/larreta.mp4',
                alt: 'descripción alternativa'
            }, buttons: BUTTONS_ARR })));
};
exports.HighLightPanel.story = { name: 'Con botones' };
exports.HighLightPanelButtonLink = function () {
    return (react_1.default.createElement("div", { className: "panel-wrapper" },
        react_1.default.createElement(_1.PanelVertical, { title: TITLE, description: DESCRIPTION, video: {
                src: 'panel/larreta.mp4',
                alt: 'descripción alternativa'
            }, link: LINK_BUTTON })));
};
exports.HighLightPanelButtonLink.story = { name: 'Con botón de descarga' };
exports.HighLightPanelLink = function () {
    return (react_1.default.createElement("div", { className: "panel-wrapper" },
        react_1.default.createElement(_1.PanelVertical, { title: TITLE, description: DESCRIPTION, video: {
                src: 'panel/larreta.mp4',
                alt: 'descripción alternativa'
            }, link: LINK })));
};
exports.HighLightPanelLink.story = { name: 'Con enlace de descarga' };
