// @gcba/obelisco/dist/components/Collapse/Collapse.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.CollapseList = exports.CollapsableLabelIcon = exports.CollapseIcon = exports.CollapseInformation = exports.CollapseMultipleSelection = exports.CollapseUniqueSelection = void 0;
// Base
var react_1 = __importDefault(require("react"));
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Colapsable',
    decorators: [addon_a11y_1.withA11y]
};
var COLLAPSE_DESCRIPTION = 'Esta es la descripción que se encuentra dentro de un colapsable. Recomendamos no utilizar demasiado texto, para generar una lectura óptima.';
exports.CollapseUniqueSelection = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion", id: "accordionExample" },
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseOne", isOnlySelect: true }, "T\u00EDtulo"),
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseTwo", isOnlySelect: true }, "T\u00EDtulo"),
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseThree", isOnlySelect: true }, "T\u00EDtulo")))); };
exports.CollapseUniqueSelection.story = { name: 'Selección única' };
exports.CollapseMultipleSelection = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion" },
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseOne" }, "T\u00EDtulo"),
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseTwo" }, "T\u00EDtulo"),
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseThree" }, "T\u00EDtulo")))); };
exports.CollapseMultipleSelection.story = { name: 'Selección múltiple' };
var CollapseContent = function () { return (react_1.default.createElement("div", { className: "card-body" },
    react_1.default.createElement("small", { className: "text-success" }, "$200,00"),
    react_1.default.createElement("h5", { className: "mb-0" }, "Solicitud de informes varios"),
    react_1.default.createElement("small", null, "Boleta N\u00B0 2549-48758798"),
    react_1.default.createElement("hr", null),
    react_1.default.createElement("small", { className: "text-success" }, "$250,00"),
    react_1.default.createElement("h5", { className: "mb-0" }, "Ecoparque"),
    react_1.default.createElement("small", null, "Boleta N\u00B0 2549-48758798"))); };
exports.CollapseInformation = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion" },
        react_1.default.createElement(_1.Collapse, { identifier: "collapseOne", content: react_1.default.createElement(CollapseContent, null) },
            react_1.default.createElement("span", { className: "collapse-label" }, "Total a pagar"),
            react_1.default.createElement("h4", { className: "collapse-title" }, "$450,00"))))); };
exports.CollapseInformation.story = { name: 'Con información' };
exports.CollapseIcon = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion" },
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseOne" },
            react_1.default.createElement("i", { className: "bx bxs-user-circle" }),
            " T\u00EDtulo")))); };
exports.CollapseIcon.story = { name: 'Con ícono' };
exports.CollapsableLabelIcon = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion" },
        react_1.default.createElement(_1.Collapse, { description: COLLAPSE_DESCRIPTION, identifier: "collapseOne" },
            react_1.default.createElement("i", { className: "bx bxs-user-circle" }),
            react_1.default.createElement("span", { className: "collapse-label" }, "Volanta"),
            react_1.default.createElement("h4", { className: "collapse-title" }, "T\u00EDtulo"),
            react_1.default.createElement("span", { className: "collapse-subtitle" }, "Descripci\u00F3n"))))); };
exports.CollapsableLabelIcon.story = { name: 'Con volanta y descripción' };
var CollapseForm = function () { return (react_1.default.createElement("form", { className: "card-body" },
    react_1.default.createElement("label", { className: "card-details", htmlFor: "visa" },
        react_1.default.createElement("i", { className: "bx bxl-visa" }),
        react_1.default.createElement("div", { className: "card-details-content" },
            react_1.default.createElement("h5", null, "Visa D\u00E9bito ****1234"),
            react_1.default.createElement("small", null, "Heraldo Paez")),
        react_1.default.createElement("input", { type: "radio", className: "form-radio", name: "credit_card", id: "visa" })),
    react_1.default.createElement("hr", null),
    react_1.default.createElement("label", { className: "card-details", htmlFor: "master" },
        react_1.default.createElement("i", { className: "bx bxl-mastercard" }),
        react_1.default.createElement("div", { className: "card-details-content" },
            react_1.default.createElement("h5", null, "MasterCard Cr\u00E9dito ****5678"),
            react_1.default.createElement("small", null, "Heraldo Paez")),
        react_1.default.createElement("input", { type: "radio", className: "form-radio", name: "credit_card", id: "master" })),
    react_1.default.createElement("hr", null),
    react_1.default.createElement("label", { className: "card-details", htmlFor: "new_card" },
        react_1.default.createElement("span", { className: "material-icons-round" }, "info"),
        react_1.default.createElement("div", { className: "card-details-content" },
            react_1.default.createElement("h5", null, "Nueva tarjeta de cr\u00E9dito")),
        react_1.default.createElement("input", { type: "radio", className: "form-radio", name: "credit_card", id: "new_card" })))); };
exports.CollapseList = function () { return (react_1.default.createElement("div", { className: "storybook__container-accordion" },
    react_1.default.createElement("div", { className: "accordion" },
        react_1.default.createElement(_1.Collapse, { identifier: "collapseOne", content: react_1.default.createElement(CollapseForm, null) },
            react_1.default.createElement("h4", { className: "collapse-title" }, "Tarjeta de cr\u00E9dito / d\u00E9bito "),
            react_1.default.createElement("span", { className: "collapse-subtitle" }, "VISA, MasterCard, AMEX, CABAL, Maestro"))))); };
exports.CollapseList.story = { name: 'Con lista seleccionable' };
