// @gcba/obelisco/dist/components/Scrollbar/Scrollbar.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.VerticalScrollbarComponent = exports.HorizontalScrollbarComponent = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Scrollbar.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Barra de desplazamiento',
    component: _1.Scrollbar,
    decorators: [addon_a11y_1.withA11y]
};
exports.HorizontalScrollbarComponent = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-scrollbar" },
        react_1.default.createElement(_1.Scrollbar, null,
            react_1.default.createElement("div", { className: "horizontal-example" },
                react_1.default.createElement("h1", null, "Esto es un t\u00EDtulo de ejemplo"),
                react_1.default.createElement("p", { className: "lead" }, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical."),
                react_1.default.createElement("br", null),
                react_1.default.createElement("hr", null),
                react_1.default.createElement("br", null),
                react_1.default.createElement("h2", null, "Esto es un t\u00EDtulo de ejemplo"),
                react_1.default.createElement("p", null, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical.")))));
};
exports.HorizontalScrollbarComponent.story = {
    name: 'Horizontal'
};
exports.VerticalScrollbarComponent = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-scrollbar" },
        react_1.default.createElement(_1.Scrollbar, null,
            react_1.default.createElement("div", { className: "vertical-example" },
                react_1.default.createElement("h1", null, "Esto es un t\u00EDtulo de ejemplo"),
                react_1.default.createElement("p", { className: "lead" }, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical."),
                react_1.default.createElement("br", null),
                react_1.default.createElement("hr", null),
                react_1.default.createElement("br", null),
                react_1.default.createElement("p", null, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical."),
                react_1.default.createElement("br", null),
                react_1.default.createElement("hr", null),
                react_1.default.createElement("br", null),
                react_1.default.createElement("h2", null, "Esto es un t\u00EDtulo de ejemplo"),
                react_1.default.createElement("p", null, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical."),
                react_1.default.createElement("br", null),
                react_1.default.createElement("hr", null),
                react_1.default.createElement("br", null),
                react_1.default.createElement("h2", null, "Esto es un t\u00EDtulo de ejemplo"),
                react_1.default.createElement("p", null, "Esto es un p\u00E1rrafo de ejemplo para probar el funcionamiento de la barra de desplazamiento. No tiene ning\u00FAn fin espec\u00EDfico, solo sirve de muestra. El ancho del texto es para probar el desplazamiento horizontal, mientras que la lista de cuerpos de contenido consecutivos es para probar el desplazamiento vertical.")))));
};
exports.VerticalScrollbarComponent.story = {
    name: 'Vertical'
};
