// @gcba/obelisco/dist/styles/Typography/Typography.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListaOrdenada = exports.Lista = exports.Citas = exports.Transformaciones = exports.TextoSecundario = exports.Parrafo = exports.Subtitulo = exports.Titulos = void 0;
// Base
var react_1 = require("react");
require("./Typography.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Estilos|Tipografía',
    decorators: [addon_a11y_1.withA11y]
};
exports.Titulos = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("h1", null, "H1 - Nunito Bold 37px"),
        react_1.default.createElement("h2", null, "H2 - Nunito Semibold 28px"),
        react_1.default.createElement("h3", null, "H3 - Nunito Semibold 21px"),
        react_1.default.createElement("h4", null, "H4 - Nunito Bold 19px"),
        react_1.default.createElement("h5", null, "H5 - Nunito Semibold 17px"),
        react_1.default.createElement("h6", null, "H6 - Open Sans Bold 16px")));
};
exports.Titulos.story = { name: 'Títulos' };
exports.Subtitulo = function () {
    return react_1.default.createElement("p", { className: "lead" }, "Open Sans Regular 19px");
};
exports.Subtitulo.story = { name: 'Subtítulo' };
exports.Parrafo = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null, "Open Sans Regular 16px"),
        react_1.default.createElement("small", null, "Open Sans Regular 13px")));
};
exports.Parrafo.story = { name: 'Párrafo' };
exports.TextoSecundario = function () {
    return react_1.default.createElement("p", { className: "text-xs" }, "Open Sans Regular 11.7px");
};
exports.TextoSecundario.story = {
    name: 'Texto secundario'
};
exports.Transformaciones = function () {
    return (react_1.default.createElement("p", null,
        "Nuestro sistema de ",
        react_1.default.createElement("strong", null, "est\u00E1ndares"),
        " brinda un ",
        react_1.default.createElement("em", null, "conjunto de herramientas"),
        " y componentes colaborativos",
        react_1.default.createElement("br", null),
        "y ",
        react_1.default.createElement("small", null, "accesibles"),
        " que define procesos para que los equipos internos y externos puedan trabajar",
        react_1.default.createElement("br", null),
        "alineadamente. De esta forma, ayudamos a crear ",
        react_1.default.createElement("mark", null, "servicios"),
        " consistent",
        react_1.default.createElement("sup", null, "es"),
        " para mejorar",
        react_1.default.createElement("br", null),
        " la experiencia digit",
        react_1.default.createElement("sub", null, "al"),
        " de vecinos y vecina",
        react_1.default.createElement("kbd", null, "s"),
        "."));
};
exports.Citas = function () {
    return (react_1.default.createElement("div", { className: "bloquote-wrapper" },
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Este es un apartado del texto que se quiere destacar por su importancia dentro del contenido de la p\u00E1gina.")),
        react_1.default.createElement("br", null),
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Este es un apartado del texto que se quiere destacar por su importancia dentro del contenido de la p\u00E1gina."),
            react_1.default.createElement("small", null, "Nombre del autor de la cita"))));
};
exports.Lista = function () {
    return (react_1.default.createElement("ul", null,
        react_1.default.createElement("li", null, "Esto es el texto de una lista"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista")));
};
exports.ListaOrdenada = function () {
    return (react_1.default.createElement("ol", null,
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada"),
        react_1.default.createElement("li", null, "Esto es el texto de una lista ordenada")));
};
exports.ListaOrdenada.story = {
    name: 'Lista ordenada'
};
