// @gcba/obelisco/dist/components/List-group/Access.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Vertical = exports.Horizontal = exports.AccessSizesStory = exports.AccessIcon = exports.AccessDescription = exports.AccessBorder = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./List-group.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var _1 = require(".");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Accesos',
    decorators: [addon_a11y_1.withA11y]
};
exports.AccessBorder = function () { return (react_1.default.createElement("div", { className: "storybook__container-list-group" },
    react_1.default.createElement("div", { className: "list-group" }, _1.AccessTypes.map(function (item, index) { return (react_1.default.createElement(_1.AccessBorderComponent, { key: index, type: item.type })); })))); };
exports.AccessBorder.story = {
    name: 'Con borde'
};
exports.AccessDescription = function () { return (react_1.default.createElement("div", { className: "storybook__container-list-group" },
    react_1.default.createElement("div", { className: "list-group" }, _1.AccessTypes.map(function (item, index) { return (react_1.default.createElement(_1.AccessDescriptionComponent, { key: index, type: item.type })); })))); };
exports.AccessDescription.story = {
    name: 'Con descripción'
};
exports.AccessIcon = function () { return (react_1.default.createElement("div", { className: "storybook__container-list-group" },
    react_1.default.createElement("div", { className: "list-group" }, _1.AccessTypes.map(function (item, index) { return (react_1.default.createElement(_1.AccessIconComponent, { key: index, type: item.type })); })))); };
exports.AccessIcon.story = {
    name: 'Con ícono'
};
exports.AccessSizesStory = function () { return (react_1.default.createElement("div", { className: "storybook__container-list-group-sizes" },
    react_1.default.createElement("div", { className: "list-group" }, _1.AccessSizes.map(function (item, index) { return (react_1.default.createElement(_1.AccessSizesComponent, { key: index, type: item.type, name: item.name })); })))); };
exports.AccessSizesStory.story = {
    name: 'Tamaños'
};
exports.Horizontal = function () { return react_1.default.createElement(_1.AccessListHorizontal, null); };
exports.Vertical = function () { return (react_1.default.createElement("div", { className: "storybook__container-list-group" },
    react_1.default.createElement(_1.AccessListVertical, null))); };
