// @gcba/obelisco/dist/components/List-group/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.AccessListVertical = exports.AccessListHorizontal = exports.AccessSizesComponent = exports.AccessIconComponent = exports.AccessDescriptionComponent = exports.AccessBorderComponent = exports.AccessSizes = exports.AccessTypes = void 0;
var react_1 = __importDefault(require("react"));
exports.AccessTypes = [
    { type: '' },
    { type: 'hover' },
    { type: 'active' },
    { type: 'focus' }
];
exports.AccessSizes = [
    { type: '', name: 'grande' },
    { type: 'item-sm', name: 'chico' }
];
exports.AccessBorderComponent = function (props) {
    var type = props.type;
    var className = ("list-group-item item-sm " + type).trim();
    return (react_1.default.createElement("a", { href: "#", className: className },
        react_1.default.createElement("span", { className: "access-title" }, "Acceso")));
};
exports.AccessDescriptionComponent = function (props) {
    var type = props.type;
    var className = ("list-group-item item-sm " + type).trim();
    return (react_1.default.createElement("a", { href: "#", className: className },
        react_1.default.createElement("div", { className: "access-content" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso"),
            react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n (Opcional)"))));
};
exports.AccessIconComponent = function (props) {
    var type = props.type;
    var className = ("list-group-item item-sm " + type).trim();
    return (react_1.default.createElement("a", { href: "#", className: className },
        react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
        react_1.default.createElement("div", { className: "access-content" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso"),
            react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n (Opcional)"))));
};
exports.AccessSizesComponent = function (props) {
    var className = ("list-group-item " + props.type).trim();
    return (react_1.default.createElement("a", { href: "#", className: className },
        react_1.default.createElement("span", { className: "access-title" },
            "Acceso ",
            props.name)));
};
exports.AccessListHorizontal = function () {
    return (react_1.default.createElement("div", { className: "list-group list-group-row" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso"))));
};
exports.AccessListVertical = function () {
    return (react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bxs-info-circle" }),
                "Acceso"))));
};
