// @gcba/obelisco/dist/components/List-group/List-group.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListaDeAccesosDescriptivo = exports.ListaDeAccesosConCategoria = exports.ListaDeAccesos = exports.AccesoChicoConDescripcion = exports.AccesoChico = exports.AccesoConIcono = exports.Accesos = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
require("./List-group.stories.scss");
var BigAccess_stories_1 = require("./BigAccess.stories");
var AccesIcon_stories_1 = require("./AccesIcon.stories");
var SmallAcces_stories_1 = require("./SmallAcces.stories");
var AccessList_stories_1 = require("./AccessList.stories");
var SmallAccessDescription_stories_1 = require("./SmallAccessDescription.stories");
var AccessListCategory_stories_1 = require("./AccessListCategory.stories");
var ListAccessDescriptive_stories_1 = require("./ListAccessDescriptive.stories");
// Configuración del general del componente
exports.default = {
    title: 'Componentes|Accesos',
    decorators: [addon_a11y_1.withA11y]
};
exports.Accesos = function () { return (react_1.default.createElement("div", null,
    react_1.default.createElement(BigAccess_stories_1.BigAccess, null))); };
exports.AccesoConIcono = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(AccesIcon_stories_1.AccesIcon, null)));
};
exports.AccesoChico = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(SmallAcces_stories_1.SmallAccess, null)));
};
exports.AccesoChicoConDescripcion = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(SmallAccessDescription_stories_1.SmallAccessDescription, null)));
};
exports.ListaDeAccesos = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(AccessList_stories_1.AccessList, null)));
};
exports.ListaDeAccesosConCategoria = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(AccessListCategory_stories_1.AccessListCategory, null)));
};
exports.ListaDeAccesosDescriptivo = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement(ListAccessDescriptive_stories_1.AccessListDescriptive, null)));
};
