// @gcba/obelisco/dist/components/List-group/List-group.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListaDeAccesosDescriptivo = exports.ListaDeAccesosConCategoria = exports.ListaDeAccesos = void 0;
// Base
var react_1 = require("react");
require("./List-group.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
var AccessList_stories_1 = require("./AccessList.stories");
var AccessListCategory_stories_1 = require("./AccessListCategory.stories");
var ListAccessDescriptive_stories_1 = require("./ListAccessDescriptive.stories");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Accesos/Lista de accesos',
    decorators: [addon_a11y_1.withA11y]
};
exports.ListaDeAccesos = function () { return react_1.default.createElement(AccessList_stories_1.AccessList, null); };
exports.ListaDeAccesos.story = {
    name: 'Lista de accesos'
};
exports.ListaDeAccesosConCategoria = function () { return react_1.default.createElement(AccessListCategory_stories_1.AccessListCategory, null); };
exports.ListaDeAccesosConCategoria.story = {
    name: 'Lista de accesos con categoría'
};
exports.ListaDeAccesosDescriptivo = function () { return react_1.default.createElement(ListAccessDescriptive_stories_1.AccessListDescriptive, null); };
exports.ListaDeAccesosDescriptivo.story = {
    name: 'Lista de accesos descriptivo'
};
