// @gcba/obelisco/dist/components/Audio/ContainerAudio.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ContainerNews = exports.ContainerAudio = exports.TitledContainer = exports.CompleteContainer = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
exports.default = {
    title: 'Componentes|Audio/Con contenedor',
    decorators: [addon_a11y_1.withA11y]
};
exports.CompleteContainer = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("h4", { className: "card-title" }, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed"),
                react_1.default.createElement("p", { className: "card-text" }, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut"),
                react_1.default.createElement("audio", { className: "audio-sm", controls: true },
                    react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                    "Your browser does not support the audio tag.")))));
};
exports.CompleteContainer.story = { name: 'Completo' };
exports.TitledContainer = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("h4", { className: "card-title" }, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed"),
                react_1.default.createElement("audio", { className: "audio-sm mt-3", controls: true },
                    react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                    "Your browser does not support the audio tag."))),
        react_1.default.createElement("br", null),
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("p", { className: "card-text" }, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut"),
                react_1.default.createElement("audio", { className: "audio-sm", controls: true },
                    react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                    "Your browser does not support the audio tag.")))));
};
exports.TitledContainer.story = { name: 'Solo titulo' };
exports.ContainerAudio = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("audio", { className: "audio-sm", controls: true },
                    react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                    "Your browser does not support the audio tag.")))));
};
exports.ContainerAudio.story = { name: 'Solo audio' };
exports.ContainerNews = function () {
    return (react_1.default.createElement("div", { className: "cards-container" },
        react_1.default.createElement("a", { className: "card", href: "#" },
            react_1.default.createElement("img", { src: "cards/evento.jpg", className: "card-img-top", alt: "descripci\u00F3n alternativa" }),
            react_1.default.createElement("audio", { className: "audio-event", controls: true },
                react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                "Your browser does not support the audio tag."),
            react_1.default.createElement("div", { className: "card-body" },
                react_1.default.createElement("div", { className: "mb-2" },
                    react_1.default.createElement("span", { className: "badge badge-info" }, "Etiqueta 1"),
                    react_1.default.createElement("span", { className: "badge badge-info" }, "Etiqueta 2")),
                react_1.default.createElement("h4", { className: "card-title" }, "T\u00EDtulo de tarjeta"),
                react_1.default.createElement("p", { className: "card-text" }, "Recomendamos en este espacio no utilizar m\u00E1s de 3 l\u00EDneas, para generar una lectura \u00F3ptima.")),
            react_1.default.createElement("div", { className: "card-footer" },
                react_1.default.createElement("small", null, "Fecha de publicaci\u00F3n: 10/10/2021")))));
};
exports.ContainerNews.story = { name: 'Noticias' };
