// @gcba/obelisco/dist/index.js
"use strict";
/*
 * Por el momento no estamos dando soporte para componentes de react.
 * Los componentes presentes en esta librería son de uso exclusivo para la documetanción y no deben ser utilizados en proyectos reales.
 */
Object.defineProperty(exports, "__esModule", { value: true });
// Components
var Badge_1 = require("./components/Badge");
Object.defineProperty(exports, "Badge", { enumerable: true, get: function () { return Badge_1.Badge; } });
var Button_1 = require("./components/Button");
Object.defineProperty(exports, "Button", { enumerable: true, get: function () { return Button_1.Button; } });
var Card_1 = require("./components/Card");
Object.defineProperty(exports, "Card", { enumerable: true, get: function () { return Card_1.Card; } });
Object.defineProperty(exports, "SimpleCard", { enumerable: true, get: function () { return Card_1.SimpleCard; } });
var Pagination_1 = require("./components/Pagination");
Object.defineProperty(exports, "Pagination", { enumerable: true, get: function () { return Pagination_1.Pagination; } });
var Nav_1 = require("./components/Nav");
Object.defineProperty(exports, "Nav", { enumerable: true, get: function () { return Nav_1.Nav; } });
