// @gcba/obelisco/dist/components/List-group/Vertical/VerticalAccess.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SmallVerticalAccessWithIcon = exports.VerticalAccessWithIcon = exports.SmallVerticalAccessDescription = exports.VerticalAccessSmall = exports.VerticalAccess = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Accesos/Vertical',
    decorators: [addon_a11y_1.withA11y]
};
exports.VerticalAccess = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item hover" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item focus" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item list-group-item-logout" },
            react_1.default.createElement("span", { className: "access-title" }, "Cerrar sesi\u00F3n "))))); };
exports.VerticalAccess.story = {
    name: 'Acceso vertical'
};
exports.VerticalAccessSmall = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm hover" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm focus" },
            react_1.default.createElement("span", { className: "access-title" }, "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm list-group-item-logout" },
            react_1.default.createElement("span", { className: "access-title" }, "Cerrar sesi\u00F3n "))))); };
exports.VerticalAccessSmall.story = {
    name: 'Acceso vertical chico'
};
exports.SmallVerticalAccessDescription = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("div", { className: "access-content" },
                react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
                react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n"))),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm hover" },
            react_1.default.createElement("div", { className: "access-content" },
                react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
                react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n"))),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm focus" },
            react_1.default.createElement("div", { className: "access-content" },
                react_1.default.createElement("span", { className: "access-title" }, "Acceso chico"),
                react_1.default.createElement("p", { className: "access-text" }, "Descripci\u00F3n")))))); };
exports.SmallVerticalAccessDescription.story = {
    name: 'Acceso vertical chico con descripción'
};
exports.VerticalAccessWithIcon = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-facebook-circle" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-instagram" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-twitter" }),
                "Acceso grande")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-youtube" }),
                "Acceso grande"))),
    react_1.default.createElement("div", { className: "group-access" },
        react_1.default.createElement("div", { className: "card-access" },
            react_1.default.createElement("div", { className: "card-access-body" },
                react_1.default.createElement("div", null,
                    react_1.default.createElement("h3", { className: "card-access-title" }, "T\u00EDtulo"),
                    react_1.default.createElement("p", { className: "card-access-text" }, "Bajada")),
                react_1.default.createElement("i", { className: "bx bxs-shield-x card-access-icon" })),
            react_1.default.createElement("a", { href: "#", className: "list-group-item card-item" },
                react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")))),
    react_1.default.createElement("div", { className: "group-access" },
        react_1.default.createElement("div", { className: "card-access" },
            react_1.default.createElement("div", { className: "card-access-body reverse" },
                react_1.default.createElement("div", null,
                    react_1.default.createElement("h3", { className: "card-access-title" }, "T\u00EDtulo"),
                    react_1.default.createElement("p", { className: "card-access-text" }, "Bajada")),
                react_1.default.createElement("i", { className: "bx bxs-shield-x card-access-icon" })),
            react_1.default.createElement("a", { href: "#", className: "list-group-item card-item" },
                react_1.default.createElement("span", { className: "access-title" }, "Acceso grande")))))); };
exports.VerticalAccessWithIcon.story = {
    name: 'Acceso vertical con ícono'
};
exports.SmallVerticalAccessWithIcon = function () { return (react_1.default.createElement("div", { className: "list-group-container" },
    react_1.default.createElement("div", { className: "list-group" },
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-facebook-circle" }),
                "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-instagram" }),
                "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-twitter" }),
                "Acceso chico")),
        react_1.default.createElement("a", { href: "#", className: "list-group-item item-sm" },
            react_1.default.createElement("span", { className: "access-title" },
                react_1.default.createElement("i", { className: "bx bx-sm align-middle mr-2 bxl-youtube" }),
                "Acceso chico"))))); };
exports.SmallVerticalAccessWithIcon.story = {
    name: 'Acceso vertical chico con ícono'
};
