// @gcba/obelisco/dist/components/Link/Link.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Acceso = exports.Descarga = exports.Externo = exports.Default = void 0;
// Base
var react_1 = require("react");
require("./Link.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
// Config
exports.default = {
    title: 'Componentes|Enlace',
    decorators: [addon_a11y_1.withA11y]
};
exports.Default = function () {
    return (react_1.default.createElement("p", null,
        "You never see a frog so modest and",
        ' ',
        react_1.default.createElement("a", { className: "", href: "#", target: "_blank", rel: "noreferrer" }, "enlace en p\u00E1rrafo"),
        ' ',
        "as he was, for all he was so gifted. And when it come to fair and square jumping on a dead level, he could get over more ground at one straddle than any animal of his breed you ever see."));
};
exports.Externo = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "You never see a frog so modest and",
            ' ',
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank", rel: "noreferrer" }, "enlace externo"),
            ' ',
            "as he was, for all he was so gifted. And when it come to fair and square jumping on a dead level, he could get over more ground at one straddle than any animal of his breed you ever see.")));
};
exports.Descarga = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "You never see a frog so modest and",
            ' ',
            react_1.default.createElement("a", { className: "download", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de descarga"),
            ' ',
            "as he was, for all he was so gifted. And when it come to fair and square jumping on a dead level, he could get over more ground at one straddle than any animal of his breed you ever see."),
        react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet consectetur adipisicing elit."),
        react_1.default.createElement("a", { className: "btn btn-secondary btn-sm download-link", href: "#", target: "_blank", rel: "noreferrer" }, "Descargar")));
};
exports.Acceso = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "You never see a frog so modest and",
            ' ',
            react_1.default.createElement("a", { className: "access", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de acceso"),
            ' ',
            "as he was, for all he was so gifted. And when it come to fair and square jumping on a dead level, he could get over more ground at one straddle than any animal of his breed you ever see.")));
};
