// @gcba/obelisco/dist/components/Link/Link.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Acceso = exports.Descarga = exports.Externo = exports.Predeterminado = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Link.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Enlace',
    decorators: [addon_a11y_1.withA11y]
};
exports.Predeterminado = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "enlace en p\u00E1rrafo"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente. Adem\u00E1s contamos con la etiqueta small para",
            ' ',
            react_1.default.createElement("small", null,
                react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer" }, "enlaces en p\u00E1rrafo"),
                ' '),
            "con menor tama\u00F1o de fuente"),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container__link" },
            react_1.default.createElement("p", null,
                "Nuestro sistema de",
                ' ',
                react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer", className: "link-white" }, "enlace en p\u00E1rrafo"),
                ' ',
                "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente. Adem\u00E1s contamos con la etiqueta small para",
                ' ',
                react_1.default.createElement("small", null,
                    react_1.default.createElement("a", { href: "#", target: "_blank", rel: "noreferrer", className: "link-white" }, "enlaces en p\u00E1rrafo"),
                    ' '),
                "con menor tama\u00F1o de fuente"))));
};
exports.Externo = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank", rel: "noreferrer" }, "enlace externo"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container__link" },
            react_1.default.createElement("p", null,
                "Nuestro sistema de",
                ' ',
                react_1.default.createElement("a", { className: "external link-white", href: "#", target: "_blank", rel: "noreferrer" }, "enlace externo"),
                ' ',
                "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."))));
};
exports.Descarga = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "download", href: "#", target: "_blank", rel: "noreferrer", download: true }, "enlace de descarga"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."),
        react_1.default.createElement("p", null, "De esta forma, ayudamos a crear servicios consistentes para mejorar la experiencia digital de vecinos y vecinas."),
        react_1.default.createElement("a", { className: "btn btn-secondary btn-sm download-link", href: "#", target: "_blank", rel: "noreferrer", download: true }, "Descargar"),
        react_1.default.createElement("br", null),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container__link" },
            react_1.default.createElement("p", null,
                "Nuestro sistema de",
                ' ',
                react_1.default.createElement("a", { className: "download link-white", href: "#", target: "_blank", rel: "noreferrer", download: true }, "enlace de descarga"),
                ' ',
                "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."),
            react_1.default.createElement("p", null, "De esta forma, ayudamos a crear servicios consistentes para mejorar la experiencia digital de vecinos y vecinas."))));
};
exports.Acceso = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "access", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de acceso"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container__link" },
            react_1.default.createElement("p", null,
                "Nuestro sistema de",
                ' ',
                react_1.default.createElement("a", { className: "access link-white", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de acceso"),
                ' ',
                "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."))));
};
