// @gcba/obelisco/dist/components/Link/Link.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Acceso = exports.Descarga = exports.Externo = exports.Predeterminado = void 0;
// Base
var react_1 = require("react");
require("./Link.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Enlace',
    decorators: [addon_a11y_1.withA11y]
};
exports.Predeterminado = function () {
    return (react_1.default.createElement("p", null,
        "Nuestro sistema de",
        ' ',
        react_1.default.createElement("a", { className: "", href: "#", target: "_blank", rel: "noreferrer" }, "enlace en p\u00E1rrafo"),
        ' ',
        "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."));
};
exports.Externo = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "external", href: "#", target: "_blank", rel: "noreferrer" }, "enlace externo"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente.")));
};
exports.Descarga = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "download", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de descarga"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente."),
        react_1.default.createElement("p", null, "De esta forma, ayudamos a crear servicios consistentes para mejorar la experiencia digital de vecinos y vecinas."),
        react_1.default.createElement("a", { className: "btn btn-secondary btn-sm download-link", href: "#", target: "_blank", rel: "noreferrer" }, "Descargar")));
};
exports.Acceso = function () {
    return (react_1.default.createElement("div", null,
        react_1.default.createElement("p", null,
            "Nuestro sistema de",
            ' ',
            react_1.default.createElement("a", { className: "access", href: "#", target: "_blank", rel: "noreferrer" }, "enlace de acceso"),
            ' ',
            "brinda un conjunto de herramientas y componentes colaborativos y accesibles que define procesos para que los equipos internos y externos puedan trabajar alineadamente.")));
};
