// @gcba/obelisco/dist/components/Audio/Audio.stories.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ConContenedor = exports.Audio = void 0;
// Base
var react_1 = __importDefault(require("react"));
require("./Audio.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Configuración general del componente
exports.default = {
    title: 'Componentes|Audio',
    decorators: [addon_a11y_1.withA11y]
};
exports.Audio = function () {
    return (react_1.default.createElement("div", { className: "storybook__container-audio" },
        react_1.default.createElement("audio", { controls: true },
            react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
            "Your browser does not support the audio tag.")));
};
exports.ConContenedor = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("div", { className: "storybook__container-cards" },
            react_1.default.createElement("div", { className: "card" },
                react_1.default.createElement("div", { className: "card-body" },
                    react_1.default.createElement("h4", { className: "card-title" }, "T\u00EDtulo de contenedor"),
                    react_1.default.createElement("p", { className: "card-text" }, "Recomendamos en este espacio no utilizar m\u00E1s de 3 l\u00EDneas, para generar una lectura \u00F3ptima."),
                    react_1.default.createElement("audio", { className: "audio-sm", controls: true },
                        react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                        "Your browser does not support the audio tag.")))),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container-cards" },
            react_1.default.createElement("div", { className: "card" },
                react_1.default.createElement("div", { className: "card-body" },
                    react_1.default.createElement("h4", { className: "card-title" }, "T\u00EDtulo de contenedor"),
                    react_1.default.createElement("audio", { className: "audio-sm mt-3", controls: true },
                        react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                        "Tu navegador no soporta la etiqueta de audio.")))),
        react_1.default.createElement("br", null),
        react_1.default.createElement("div", { className: "storybook__container-cards" },
            react_1.default.createElement("div", { className: "card" },
                react_1.default.createElement("div", { className: "card-body" },
                    react_1.default.createElement("audio", { className: "audio-sm", controls: true },
                        react_1.default.createElement("source", { src: "https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1", type: "audio/ogg" }),
                        "Tu navegador no soporta la etiqueta de audio."))))));
};
exports.ConContenedor.story = {
    name: 'Con contenedor'
};
/*
export const ContainerNews = (): JSX.Element => {
  return (
    <div className="storybook__container-cards">
      <a className="card" href="#">
        <img src="cards/evento.jpg" className="card-img-top" alt="descripción alternativa" />
        <audio className="audio-event" controls>
          <source
            src="https://www.zapsplat.com/wp-content/uploads/2015/sound-effects-61905/zapsplat_multimedia_alert_chime_short_musical_notification_cute_child_like_001_64918.mp3?_=1"
            type="audio/ogg"
          />
          Tu navegador no soporta la etiqueta de audio.
        </audio>
        <div className="card-body">
          <div className="mb-2">
            <span className="badge badge-info">Etiqueta 1</span>
            <span className="badge badge-info">Etiqueta 2</span>
          </div>
          <h4 className="card-title">Título de tarjeta</h4>
          <p className="card-text">
            Recomendamos en este espacio no utilizar más de 3 líneas, para generar una lectura óptima.
          </p>
        </div>
        <div className="card-footer">
          <small>Fecha de publicación: 10/10/2021</small>
        </div>
      </a>
    </div>
  );
};

ContainerNews.story = { name: 'Noticias' };
*/
