// @gcba/obelisco/dist/components/Grid/Grid.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Desplazamientos = exports.OrdenarClases = exports.AlineacionHorizontal = exports.Divisiones = exports.Columnas = void 0;
// Base
var react_1 = require("react");
require("./Grid.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
// Config
exports.default = {
    title: 'Componentes|Grilla',
    decorators: [addon_a11y_1.withA11y]
};
/*
export const Tamanos = (): JSX.Element => {
  return (
    <div className="container text-center">
      <div className="row">
        <div className="col-2"></div>
        <div className="col-2">
          <strong>Extra Pequeño</strong>
        </div>
        <div className="col-2">
          <strong>Pequeño</strong><br></br>
          <small>≥576px</small>
        </div>
        <div className="col-2">
          <strong>Medio</strong><br></br>
          <small>≥768px</small>
        </div>
        <div className="col-2">
          <strong>Grande</strong><br></br>
          <small>≥992px</small>
        </div>
        <div className="col-2">
          <strong>Extra Grande</strong><br></br>
          <small>≥1200px</small>
        </div>
      </div>
      <div className="row">
        <div className="col">Ancho máximo del contenedor</div>
        <div className="col">Ninguno (automático)</div>
        <div className="col">540 px</div>
        <div className="col">720px</div>
        <div className="col">960 px</div>
        <div className="col">1140 px</div>
      </div>
      <div className="row">
        <div className="col">Prefijo de clase	</div>
        <div className="col">.col-</div>
        <div className="col">.col-sm-</div>
        <div className="col">.col-md-</div>
        <div className="col">.col-lg-</div>
        <div className="col">.col-xl-</div>
      </div>
    </div>
  );
};

Tamanos.story = { name: 'Tamaños' };*/
exports.Columnas = function () {
    return (react_1.default.createElement("div", { className: "grid-wraper" },
        react_1.default.createElement("div", { className: "container text-center" },
            react_1.default.createElement("div", { className: "row" },
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "1")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "2")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "3")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "4")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "5")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "6")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "7")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "8")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "9")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "10")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "11")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "12"))))));
};
exports.Divisiones = function () {
    return (react_1.default.createElement("div", { className: "grid-wraper" },
        react_1.default.createElement("div", { className: "container text-center" },
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-2" },
                    react_1.default.createElement("div", null, "col-2")),
                react_1.default.createElement("div", { className: "col-6" },
                    react_1.default.createElement("div", null, "col-6")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "col-4"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-5" },
                    react_1.default.createElement("div", null, "col-5")),
                react_1.default.createElement("div", { className: "col-7" },
                    react_1.default.createElement("div", null, "col-7"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "col-4")),
                react_1.default.createElement("div", { className: "col-8" },
                    react_1.default.createElement("div", null, "col-8"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-9" },
                    react_1.default.createElement("div", null, "col-9")),
                react_1.default.createElement("div", { className: "col-3" },
                    react_1.default.createElement("div", null, "col-3"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "col-1")),
                react_1.default.createElement("div", { className: "col-10" },
                    react_1.default.createElement("div", null, "col-10")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "col-1"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-11" },
                    react_1.default.createElement("div", null, "col-11")),
                react_1.default.createElement("div", { className: "col-1" },
                    react_1.default.createElement("div", null, "col-1"))))));
};
exports.AlineacionHorizontal = function () {
    return (react_1.default.createElement("div", { className: "grid-wraper" },
        react_1.default.createElement("div", { className: "container" },
            react_1.default.createElement("div", { className: "row justify-content-start mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas"))),
            react_1.default.createElement("div", { className: "row justify-content-center mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas"))),
            react_1.default.createElement("div", { className: "row justify-content-end mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas"))),
            react_1.default.createElement("div", { className: "row justify-content-around mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas"))),
            react_1.default.createElement("div", { className: "row justify-content-between mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Una de dos columnas"))))));
};
exports.AlineacionHorizontal.story = { name: 'Alineación Horizontal' };
exports.OrdenarClases = function () {
    return (react_1.default.createElement("div", { className: "grid-wraper" },
        react_1.default.createElement("div", { className: "container" },
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Primero, pero desordenado")),
                react_1.default.createElement("div", { className: "col-4 order-12" },
                    react_1.default.createElement("div", null, "Tercero, pero primero")),
                react_1.default.createElement("div", { className: "col-4 order-1" },
                    react_1.default.createElement("div", null, "Segundo, pero \u00FAltimo"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-4 order-last" },
                    react_1.default.createElement("div", null, "Tercero, pero primero")),
                react_1.default.createElement("div", { className: "col-4" },
                    react_1.default.createElement("div", null, "Segundo, pero desordenado")),
                react_1.default.createElement("div", { className: "col-4 order-first" },
                    react_1.default.createElement("div", null, "Primero pero ultimo"))))));
};
exports.Desplazamientos = function () {
    return (react_1.default.createElement("div", { className: "grid-wraper" },
        react_1.default.createElement("div", { className: "container" },
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-md-4" },
                    react_1.default.createElement("div", null, ".col-md-4")),
                react_1.default.createElement("div", { className: "col-md-4 offset-md-4" },
                    react_1.default.createElement("div", null, ".col-md-4 .offset-md-4"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-md-3 offset-md-3" },
                    react_1.default.createElement("div", null, ".col-md-3 .offset-md-3")),
                react_1.default.createElement("div", { className: "col-md-3 offset-md-3" },
                    react_1.default.createElement("div", null, ".col-md-3 .offset-md-3"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-md-6 offset-md-3" },
                    react_1.default.createElement("div", null, ".col-md-6 .offset-md-3"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-sm-5 col-md-6" },
                    react_1.default.createElement("div", null, ".col-sm-5 .col-md-6")),
                react_1.default.createElement("div", { className: "col-sm-5 offset-sm-2 col-md-6 offset-md-0" },
                    react_1.default.createElement("div", null, ".col-sm-5 .offset-sm-2 .col-md-6 .offset-md-0"))),
            react_1.default.createElement("div", { className: "row mb-2" },
                react_1.default.createElement("div", { className: "col-sm-6 col-md-5 col-lg-6" },
                    react_1.default.createElement("div", null, ".col-sm-6 .col-md-5 .col-lg-6")),
                react_1.default.createElement("div", { className: "col-sm-6 col-md-5 offset-md-2 col-lg-6 offset-lg-0" },
                    react_1.default.createElement("div", null, ".col-sm-6 .col-md-5 .offset-md-2 .col-lg-6 .offset-lg-0"))))));
};
