// @gcba/obelisco/dist/components/StatusMessage/StatusMessage.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.StatusInstitutional = exports.StatusDanger = exports.StatusInfo = exports.StatusWarning = exports.StatusSuccess = void 0;
// Base
var react_1 = require("react");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
var _1 = require(".");
require("./status.stories.scss");
// Configuración del general del componente
exports.default = {
    title: 'Componentes|Mensajes de estado',
    decorators: [addon_a11y_1.withA11y]
};
var LIST_SUCCCESS = [
    'Éste es un ítem dentro de una lista que puede contener un <a href="#">enlace</a>.',
    'Los ítems pueden contener o no enlaces. Este es un ejemplo para un detalle que no lleva enlace.',
    'Éste es otro ítem dentro de una lista.'
];
var LIST = [
    'Contar con Clave Ciudad de <a href="#">AGIP</a>',
    'Cargar en el Sistema de Tramitación a Distancia (TAD) la siguiente documentación obligatoria:',
    'Anexo II Formulario de Presentación de Programa de Capacitación'
];
exports.StatusSuccess = function () { return (react_1.default.createElement("div", { className: "status-container" },
    react_1.default.createElement(_1.StatusMessage, { title: "Esto es un t\u00EDtulo de un mensaje de \u00E9xito", type: "success", text: "Esto es un p\u00E1rrafo que acompa\u00F1a y sirve para indicar que el usuario realiz\u00F3 una acci\u00F3n correctamente, como cuando realiza una tarea con \u00E9xito.", list: LIST_SUCCCESS }))); };
exports.StatusSuccess.story = {
    name: 'Éxito'
};
exports.StatusWarning = function () { return (react_1.default.createElement("div", { className: "status-container" },
    react_1.default.createElement(_1.StatusMessage, { title: "Esto es un t\u00EDtulo de un mensaje de advertencia", type: "primary", text: "Esto es un p\u00E1rrafo que advierte y sirve para indicarle al usuario que preste atenci\u00F3n a determinada informaci\u00F3n o acci\u00F3n que debe realizar.", list: LIST, isTwoButtons: true }))); };
exports.StatusWarning.story = {
    name: 'Advertencia'
};
exports.StatusInfo = function () { return (react_1.default.createElement("div", { className: "status-container" },
    react_1.default.createElement(_1.StatusMessage, { title: "Esto es un t\u00EDtulo de un mensaje de informaci\u00F3n", type: "info", text: [
            'Este es un párrafo que brinda información necesaria con la finalidad de guiar y ayudar a las personas usuarias a finalizar un proceso o iniciar uno nuevo.',
            'Este es un párrafo que brinda información necesaria con la finalidad de guiar y ayudar a las personas usuarias a finalizar un proceso o iniciar uno nuevo.'
        ] }))); };
exports.StatusInfo.story = {
    name: 'Información'
};
exports.StatusDanger = function () { return (react_1.default.createElement("div", { className: "status-container" },
    react_1.default.createElement(_1.StatusMessage, { title: "Esto es un t\u00EDtulo de un mensaje de peligro", type: "danger", text: "Este p\u00E1rrafo sirve para comunicar que algo sali\u00F3 mal y explica c\u00F3mo puede ser solucionado. Muestra errores sobre algo que se realiz\u00F3.", list: LIST, isTwoButtons: true }))); };
exports.StatusDanger.story = {
    name: 'Error'
};
exports.StatusInstitutional = function () { return (react_1.default.createElement("div", { className: "status-container-institutional" },
    react_1.default.createElement(_1.StatusMessage, { title: "Este es un mensaje institucional del Gobierno de la Ciudad de Buenos Aires", type: "institutional", text: "En el cuerpo de este componente pueden ir enlaces o botones: <a href='#'>https://ejemplo.buenosaires.gob.ar</a>" }))); };
exports.StatusInstitutional.story = {
    name: 'Institucional'
};
