// @gcba/obelisco/dist/components/Typography/Typography.stories.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ListaOrdenada = exports.Lista = exports.Citas = exports.Transformaciones = exports.Párrafo = exports.Subtítulo = exports.Títulos = void 0;
// Base
var react_1 = require("react");
require("./Typography.stories.scss");
// Addons
var addon_a11y_1 = require("@storybook/addon-a11y");
// Components
// Config
exports.default = {
    title: 'Componentes|Tipografía',
    decorators: [addon_a11y_1.withA11y]
};
var demoText = 'Lorem, ipsum dolor sit amet consectetur adipisicing elit.';
exports.Títulos = function () {
    return (react_1.default.createElement(react_1.default.Fragment, null,
        react_1.default.createElement("h1", null,
            "h1 - ",
            demoText),
        react_1.default.createElement("h2", null,
            "h2 - ",
            demoText),
        react_1.default.createElement("h3", null,
            "h3 - ",
            demoText),
        react_1.default.createElement("h4", null,
            "h4 - ",
            demoText),
        react_1.default.createElement("h5", null,
            "h5 - ",
            demoText),
        react_1.default.createElement("h6", null,
            "h6 - ",
            demoText)));
};
exports.Subtítulo = function () {
    return react_1.default.createElement("p", { className: "lead" }, demoText);
};
exports.Párrafo = function () {
    return (react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias omnis similique, placeat accusamus porro voluptates tempora provident dolorem, eius dolore facilis. Ducimus facilis suscipit quaerat amet consectetur nobis, unde numquam?"));
};
exports.Transformaciones = function () {
    return (react_1.default.createElement("p", null,
        "Lorem ipsum dolor sit ",
        react_1.default.createElement("strong", null, "negrita"),
        " consectetur adipisicing",
        react_1.default.createElement("em", null, "cursiva"),
        " elit. Molestias ",
        react_1.default.createElement("small", null, "peque\u00F1o"),
        " omnis",
        react_1.default.createElement("br", null),
        "similique ",
        react_1.default.createElement("mark", null, "remarcado"),
        ", place",
        react_1.default.createElement("sup", null, "at"),
        " accusamus porro voluptat",
        react_1.default.createElement("sub", null, "es"),
        " tempora provident dolorem ",
        react_1.default.createElement("kbd", null, "p"),
        "."));
};
exports.Citas = function () {
    return (react_1.default.createElement("div", { className: "bloquote-wrapper" },
        react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua."),
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut dolore.")),
        react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua."),
        react_1.default.createElement("blockquote", null,
            react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut dolore."),
            react_1.default.createElement("small", null, "Autor (no obligatorio)")),
        react_1.default.createElement("p", null, "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.")));
};
exports.Lista = function () {
    return (react_1.default.createElement("ul", null,
        react_1.default.createElement("li", null, demoText),
        react_1.default.createElement("li", null, demoText),
        react_1.default.createElement("li", null, demoText)));
};
exports.ListaOrdenada = function () {
    return (react_1.default.createElement("ol", null,
        react_1.default.createElement("li", null, demoText),
        react_1.default.createElement("li", null, demoText),
        react_1.default.createElement("li", null, demoText)));
};
